
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.04fe9e55f2b2.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.aafaa535cb7f.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.ca6dfa7b90b5.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs2.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-pnas.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Toward a theory of evolution as multilevel learning - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C4B52EA692EC50000521D28602153.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="1.6.0.post2+24fead6" />

<meta name="ncbi_domain" content="pnas" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC8833143/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8833143/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Proceedings of the National Academy of Sciences of the United States of America" /><meta name="citation_title" content="Toward a theory of evolution as multilevel learning" /><meta name="citation_author" content="Vitaly Vanchurin" /><meta name="citation_author_institution" content="aNational Center for Biotechnology Information, National Library of Medicine, Bethesda, MD 20894;" /><meta name="citation_author_institution" content="bDuluth Institute for Advanced Study, Duluth, MN 55804;" /><meta name="citation_author" content="Yuri I. Wolf" /><meta name="citation_author_institution" content="aNational Center for Biotechnology Information, National Library of Medicine, Bethesda, MD 20894;" /><meta name="citation_author" content="Mikhail I. Katsnelson" /><meta name="citation_author_institution" content="cInstitute for Molecules and Materials, Radboud University, Nijmegen 6525AJ, The Netherlands" /><meta name="citation_author" content="Eugene V. Koonin" /><meta name="citation_author_institution" content="aNational Center for Biotechnology Information, National Library of Medicine, Bethesda, MD 20894;" /><meta name="citation_publication_date" content="2022/02/02" /><meta name="citation_issue" content="6" /><meta name="citation_volume" content="119" /><meta name="citation_doi" content="10.1073/pnas.2120037119" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC8833143/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC8833143/" /><meta name="citation_pmid" content="35121666" /><meta name="DC.Title" content="Toward a theory of evolution as multilevel learning" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="National Academy of Sciences" /><meta name="DC.Contributor" content="Vitaly Vanchurin" /><meta name="DC.Contributor" content="Yuri I. Wolf" /><meta name="DC.Contributor" content="Mikhail I. Katsnelson" /><meta name="DC.Contributor" content="Eugene V. Koonin" /><meta name="DC.Date" content="2022 Feb 8" /><meta name="DC.Identifier" content="10.1073/pnas.2120037119" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Toward a theory of evolution as multilevel learning" /><meta property="og:type" content="article" /><meta property="og:description" content="Modern evolutionary theory gives a detailed quantitative description of microevolutionary processes that occur within evolving populations of organisms, but evolutionary transitions and emergence of multiple levels of complexity remain poorly understood. ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8833143/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/400840/">v.119(6); 2022 Feb 8</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC8833143
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pubreader-link other_item"><a href="/pmc/articles/PMC8833143/?report=reader" class="sidefm-pmclink">PubReader</a></li>
<li class="pdf-link other_item"><a href="/pmc/articles/PMC8833143/pdf/pnas.202120037.pdf" class="int-view">PDF (692K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/8833143/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/8833143/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="8833143" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8833143%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="cVDeASjr1dUZ87Mozp5Ut2SC4P2s2Pec3i4hl7Ra9RXJj2djuBVaCqBNtflx6yoE">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8833143%2F&amp;text=Toward%20a%20theory%20of%20evolution%20as%20multilevel%20learning" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8833143%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8833143/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC8833143/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/35121666/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC8833143/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/35121666/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC8833143/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/8833143/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2/">Proc Natl Acad Sci U S A</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/400840/">v.119(6); 2022 Feb 8</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC8833143
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-pnas.png" alt="Logo of pnas" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.pnas.org" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Proc Natl Acad Sci U S A.</span> 2022 Feb 8; 119(6): e2120037119. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2022 Feb 4. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1073%2Fpnas.2120037119" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1073/pnas.2120037119</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC8833143</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/35121666">35121666</a></div></div></div><h1 class="content-title">Toward a theory of evolution as multilevel learning</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Vanchurin%20V%5BAuthor%5D" class="affpopup" co-rid="_co_idm140580307608016" co-class="co-affbox">Vitaly Vanchurin</a>,<sup>
a
,</sup><sup>
b
,</sup><sup>
1
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wolf%20YI%5BAuthor%5D" class="affpopup" co-rid="_co_idm140580309491968" co-class="co-affbox">Yuri I. Wolf</a>,<sup>
a
</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Katsnelson%20MI%5BAuthor%5D" class="affpopup" co-rid="_co_idm140580351823152" co-class="co-affbox">Mikhail I. Katsnelson</a>,<sup>
c
</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Koonin%20EV%5BAuthor%5D" class="affpopup" co-rid="_co_idm140580307332464" co-class="co-affbox">Eugene V. Koonin</a><sup>
a
,</sup><sup>
1
</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140580307608016"><h3 class="no_margin">Vitaly Vanchurin</h3><p><sup>a</sup>National Center for Biotechnology Information, National Library of Medicine, Bethesda, MD 20894;</p><p><sup>b</sup>Duluth Institute for Advanced Study, Duluth, MN 55804;</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Vanchurin%20V%5BAuthor%5D">Vitaly Vanchurin</a></div></div><div id="_co_idm140580309491968"><h3 class="no_margin">Yuri I. Wolf</h3><p><sup>a</sup>National Center for Biotechnology Information, National Library of Medicine, Bethesda, MD 20894;</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wolf%20YI%5BAuthor%5D">Yuri I. Wolf</a></div></div><div id="_co_idm140580351823152"><h3 class="no_margin">Mikhail I. Katsnelson</h3><p><sup>c</sup>Institute for Molecules and Materials, Radboud University, Nijmegen 6525AJ, The Netherlands</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Katsnelson%20MI%5BAuthor%5D">Mikhail I. Katsnelson</a></div></div><div id="_co_idm140580307332464"><h3 class="no_margin">Eugene V. Koonin</h3><p><sup>a</sup>National Center for Biotechnology Information, National Library of Medicine, Bethesda, MD 20894;</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Koonin%20EV%5BAuthor%5D">Eugene V. Koonin</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140580351850080_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140580351850080_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140580351850080_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140580351850080_ai" style="display:none"><div class="fm-affl" id="aff1"><sup>a</sup>National Center for Biotechnology Information, National Library of Medicine, Bethesda, MD 20894;</div><div class="fm-affl" id="aff2"><sup>b</sup>Duluth Institute for Advanced Study, Duluth, MN 55804;</div><div class="fm-affl" id="aff3"><sup>c</sup>Institute for Molecules and Materials, Radboud University, Nijmegen 6525AJ, The Netherlands</div><div id="cor1"><sup>1</sup>To whom correspondence may be addressed. Email: <a href="mailto:dev@null" data-email="moc.liamg@niruhcnav.ylativ" class="oemail">moc.liamg@niruhcnav.ylativ</a> or <a href="mailto:dev@null" data-email="vog.hin.mln.ibcn@ninook" class="oemail">vog.hin.mln.ibcn@ninook</a>.</div><div id="fn1">Contributed by Eugene V. Koonin; received November 2, 2021; accepted January 3, 2022; reviewed by Steven Frank and E&#x000f6;rs Szathm&#x000e1;ry</div><div><p class="p p-first-last">Author contributions: V.V., Y.I.W., M.I.K., and E.V.K. designed research; V.V. performed research; and V.V., Y.I.W., and E.V.K. wrote the paper.</p></div></div><div class="fm-article-notes hide half_rhythm" id="idm140580351850080_an" style="display:none"><div class="fm-pubdate half_rhythm">Accepted 2022 Jan 3.</div></div><div class="permissions half_rhythm hide" id="idm140580351850080_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a>  &#x000a9; 2022 the Author(s). Published by PNAS.</div><div class="license half_rhythm">This open access article is distributed under <a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">Creative Commons Attribution License 4.0 (CC BY)</a>.</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>Supplementary File.</strong></div><div><a href="/pmc/articles/PMC8833143/bin/pnas.2120037119.sapp.pdf" data-ga-action="click_feat_suppl">pnas.2120037119.sapp.pdf</a><span style="color:gray"> (580K)</span></div><div class="small guid">GUID: FC05BE8F-B101-491F-8107-FD1EC8F9447E</div></div></dd></dl><dl data-length="39" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">There are no data underlying this work.</p></dd></dl></div><div id="abstract-a.d.b.q" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.d.b.qtitle" style="text-transform: uppercase;">Significance</h2><!--article-meta--><div><p class="p p-first-last">Modern evolutionary theory gives a detailed quantitative description of microevolutionary processes that occur within evolving populations of organisms, but evolutionary transitions and emergence of multiple levels of complexity remain poorly understood. Here, we establish the correspondence among the key features of evolution, learning dynamics, and renormalizability of physical theories to outline a theory of evolution that strives to incorporate all evolutionary processes within a unified mathematical framework of the theory of learning. According to this theory, for example, replication of genetic material and natural selection readily emerge from the learning dynamics, and in sufficiently complex systems, the same learning phenomena occur on multiple levels or on different scales, similar to the case of renormalizable physical theories.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">theory of learning, loss function, natural selection, major evolutionary transitions, origin of life</span></div></div><div id="abstract-a.d.b.r" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.d.b.rtitle" style="text-transform: uppercase;">Abstract</h2><!--article-meta--><div><p class="p p-first-last">We apply the theory of learning to physically renormalizable systems in an attempt to outline a theory of biological evolution, including the origin of life, as multilevel learning. We formulate seven fundamental principles of evolution that appear to be necessary and sufficient to render a universe observable and show that they entail the major features of biological evolution, including replication and natural selection. It is shown that these cornerstone phenomena of biology emerge from the fundamental features of learning dynamics such as the existence of a loss function, which is minimized during learning. We then sketch the theory of evolution using the mathematical framework of neural networks, which provides for detailed analysis of evolutionary phenomena. To demonstrate the potential of the proposed theoretical framework, we derive a generalized version of the Central Dogma of molecular biology by analyzing the flow of information during learning (back propagation) and predicting (forward propagation) the environment by evolving organisms. The more complex evolutionary phenomena, such as major transitions in evolution (in particular, the origin of life), have to be analyzed in the thermodynamic limit, which is described in detail in the paper by Vanchurin et&#x000a0;al. [V. Vanchurin, Y. I. Wolf, E. V. Koonin, M. I. Katsnelson, <em>Proc. Natl. Acad. Sci. U.S.A.</em> 119, 10.1073/pnas.2120042119 (2022)].</p></div></div><div id="body-a.e" class="tsec sec"><h2 class="headless nomenu"></h2><p class="p p-first">What is life? If this question is asked in the scientific rather than in the philosophical context, a satisfactory answer should assume the form of a theoretical model of the origin and evolution of complex systems that are identified with life (<a href="#r1" rid="r1" class=" bibr popnode">1</a>). NASA has operationally defined life as follows: &#x0201c;Life is a self-sustaining chemical system capable of Darwinian evolution&#x0201d; (<a href="#r2" rid="r2" class=" bibr popnode">2</a>, <a href="#r3" rid="r3" class=" bibr popnode">3</a>). Apart from the insistence on chemistry, long-term evolution that involves (random) mutation, diversification, and adaptation is, indeed, an intrinsic, essential feature of life that is not apparent in any other natural phenomena. The problem with this definition, however, is that natural (Darwinian) selection itself appears to be a complex rather than an elementary phenomenon (<a href="#r4" rid="r4" class=" bibr popnode">4</a>). In all evolving organisms we are aware of, for natural selection to kick off and to sustain long-term evolution, an essential condition is replication of a complex digital information carrier (a DNA or RNA molecule). The replication fidelity must be sufficiently high to provide for the differential replication of emerging mutants and survival of the fittest ones (this replication fidelity level is often referred to as Eigen threshold) (<a href="#r5" rid="r5" class=" bibr popnode">5</a>). In modern organisms, accurate replication is ensured by elaborate molecular machineries that include not only replication and repair enzymes but also, the entire metabolic network of the cell, which supplies energy and building blocks for replication. Thus, the origin of life is a typical chicken-and-egg problem (or catch-22); accurate replication is essential for evolution, but the mechanisms ensuring replication fidelity are themselves products of complex evolutionary processes (<a href="#r6" rid="r6" class=" bibr popnode">6</a>, <a href="#r7" rid="r7" class=" bibr popnode">7</a>).</p><p>Because genome replication that underlies natural selection is itself a product of evolution, origin of life has to be explained outside of the traditional framework of evolutionary biology. Modern evolutionary theory, steeped in population genetics, gives a detailed and arguably, largely satisfactory account of microevolutionary processes: that is, evolution of allele frequencies in a population of organisms under selection and random genetic drift (<a href="#r8" rid="r8" class=" bibr popnode">8</a>, <a href="#r9" rid="r9" class=" bibr popnode">9</a>). However, this theory has little to say about the actual history of life, especially the emergence of new levels of biological complexity, and nothing at all about the origin of life.</p><p>The crucial feature of biological complexity is its hierarchical organization. Indeed, multilevel hierarchies permeate biology: from small molecules to macromolecules; from macromolecules to functional complexes, subcellular compartments, and cells; from unicellular organisms to communities, consortia, and multicellularity; from simple multicellular organisms to highly complex forms with differentiated tissues; and from organisms to communities and eventually, to eusociality and to complex biocenoses involved in biogeochemical processes on the planetary scale. All these distinct levels jointly constitute the hierarchical organization of the biosphere. Understanding the origin and evolution of this hierarchical complexity, arguably, is one of the principal goals of biology.</p><p>In large part, evolution of the multilevel organization of biological systems appears to be driven by solving optimization problems, which entails conflicts or trade-offs between optimization criteria at different levels or scales, leading to frustrated states, in the language of physics (<a href="#r10" rid="r10" class=" bibr popnode">10</a><a href="#r11" rid="r11" class=" bibr popnode"></a>&#x02013;<a href="#r12" rid="r12" class=" bibr popnode">12</a>). Two notable cases in point are parasite&#x02013;host arms races that permeate biological evolution and makes major contributions to the diversity and complexity of life-forms (<a href="#r13" rid="r13" class=" bibr popnode">13</a><a href="#r14" rid="r14" class=" bibr popnode"></a><a href="#r15" rid="r15" class=" bibr popnode"></a>&#x02013;<a href="#r16" rid="r16" class=" bibr popnode">16</a>) and multicellular organization of complex organisms, where the tendency of individual cells to reproduce at the highest possible rate is countered by the control of cell division imposed at the organismal level (<a href="#r17" rid="r17" class=" bibr popnode">17</a>, <a href="#r18" rid="r18" class=" bibr popnode">18</a>).</p><p>Two tightly linked but distinct fundamental concepts that lie effectively outside the canonical narrative of evolutionary biology address evolution of biological complexity: major transitions in evolution (MTEs) (<a href="#r19" rid="r19" class=" bibr popnode">19</a><a href="#r20" rid="r20" class=" bibr popnode"></a>&#x02013;<a href="#r21" rid="r21" class=" bibr popnode">21</a>) and multilevel selection (MLS) (<a href="#r22" rid="r22" class=" bibr popnode">22</a><a href="#r23" rid="r23" class=" bibr popnode"></a><a href="#r24" rid="r24" class=" bibr popnode"></a><a href="#r25" rid="r25" class=" bibr popnode"></a><a href="#r26" rid="r26" class=" bibr popnode"></a>&#x02013;<a href="#r27" rid="r27" class=" bibr popnode">27</a>). Each MTE involves the emergence of a new level of organization, often described as an evolutionary transition in individuality. A clear-cut example is the evolution of multicellularity, whereby a new level of selection emerges, namely selection among ensembles of cells rather than among individual cells. Multicellular life-forms (even counting only complex organisms with multiple cell types) evolved on many independent occasions during the evolution of life (<a href="#r28" rid="r28" class=" bibr popnode">28</a>, <a href="#r29" rid="r29" class=" bibr popnode">29</a>), implying that emergence of new levels of complexity is a major evolutionary trend rather than a rare, chance event.</p><p>The MLS remains a controversial concept, presumably because of the link to the long-debated subject of group selection (<a href="#r27" rid="r27" class=" bibr popnode">27</a>, <a href="#r30" rid="r30" class=" bibr popnode">30</a>). However, as a defining component of MTE, MLS appears to be indispensable. A proposed general mechanism behind the MTE, formulated by analogy with the physical theory of the origin of patterns (for example, in glass-like systems), involves competing interactions at different levels and the frustrated states, such interactions cause (<a href="#r12" rid="r12" class=" bibr popnode">12</a>). In the physical theory of spin glasses, frustrations result in nonergodicity and enable formation and persistence of long-term memory: that is, history (<a href="#r31" rid="r31" class=" bibr popnode">31</a>, <a href="#r32" rid="r32" class=" bibr popnode">32</a>). By contrast, ergodic systems have no true history because they reach all possible states during their evolution (at least in the large time limit), and thus, the only content of quasihistory of such systems is the transition from less probable to more probable states for purely combinatorial reasons: that is, entropy increase (<a href="#r33" rid="r33" class=" bibr popnode">33</a>). As emphasized in Schroedinger&#x02019;s seminal book (<a href="#r34" rid="r34" class=" bibr popnode">34</a>), even if only in general terms, life is based on &#x0201c;negentropic&#x0201d; processes, and frustrations at different levels are necessary for these processes to take off and persist (<a href="#r12" rid="r12" class=" bibr popnode">12</a>).</p><p>The origin of cells, which can and probably should be equated with the origin of life, was the first and most momentous transition at the onset of biological evolution, and as such, it is outside the purview of evolutionary biology sensu stricto. Arguably, the theoretical investigation of the origin of life can be feasible only within the framework of an envelope theory that would incorporate biological evolution as a special case. It is natural to envisage such a theory as encompassing all nonergodic processes occurring in the universe, of which life is a special case, emerging under conditions that remain to be investigated and defined.</p><p>Here, in pursuit of a maximally general theory of evolution, we adopt the formalism of the theory of machine learning (<a href="#r35" rid="r35" class=" bibr popnode">35</a>). Importantly, learning here is perceived in the maximally general sense as an objective process that occurs in all evolving systems, including but not limited to biological ones (<a href="#r36" rid="r36" class=" bibr popnode">36</a>). As such, the analogy between learning and selection appears obvious. Both types of processes involve trial and error and acceptance or rejection of the results based on some formal criteria; in other words, both are optimization processes (<a href="#r22" rid="r22" class=" bibr popnode">22</a>, <a href="#r37" rid="r37" class=" bibr popnode">37</a>, <a href="#r38" rid="r38" class=" bibr popnode">38</a>). Here, we assess how far this analogy extends by establishing the correspondence between key features of biological evolution and concepts as well as the mathematical formalism of learning theory. We make the case that loss function, which is central to the learning theory, can be usefully and generally employed as the equivalent of the fitness function in the context of evolution. Our original motivation was to explain major features of biological evolution from more general principles of physics. However, after formulating such principles and embedding them within the mathematical framework of learning, we find that the theory can potentially apply to the entire history of the evolving universe (<a href="#r36" rid="r36" class=" bibr popnode">36</a>), including physical processes that have been taking place since the big bang and chemical processes that directly antedated and set the stage for the origin of life. The central propositions of the evolution theory outlined here include both key physical principles (namely, hierarchy of scale, frequency gaps, and renormalizability) (<a href="#r39" rid="r39" class=" bibr popnode">39</a>, <a href="#r40" rid="r40" class=" bibr popnode">40</a>) and major features of life (such as MLS, persistence of genetic parasites, and programmed cell death).</p><p>We show that learning in a complex environment leads to separation of scales, with trainable variables splitting into at least two classes: faster- and slower-changing ones. Such separation of scales underlies all processes that involve the formation of complex structure in the universe from the scale of an atom to that of clusters of galaxies. We argue that, for the emergence of life, at least three temporal scales, which correspond to environmental, phenotypic, and genotypic variables, are essential. In evolving learning systems, the slowest-changing variables are digitized and acquire the replication capacity, resulting in differential reproduction depending on the loss (fitness) function value, which is necessary and sufficient for the onset of evolution by natural selection. Subsequent evolution of life involves emergence of many additional scales, which correspond to MTE. Hereafter, we use the term &#x0201c;evolution&#x0201d; to describe temporal changes of living and lifelike and prebiotic systems (organisms), whereas the more general term &#x0201c;dynamics&#x0201d; refers to temporal processes in other physical systems.</p><p class="p">At least since the publication of Schroedinger&#x02019;s book, the possibility has been discussed that, although life certainly obeys the laws of physics, a different class of laws unique to biology could exist. Often, this putative physics of life is associated with emergence (<a href="#r41" rid="r41" class=" bibr popnode">41</a><a href="#r42" rid="r42" class=" bibr popnode"></a>&#x02013;<a href="#r43" rid="r43" class=" bibr popnode">43</a>), but the nature of the involved emergent phenomena, to our knowledge, has not been clarified until very recently (<a href="#r36" rid="r36" class=" bibr popnode">36</a>). Here, we outline a general approach to modeling and studying evolution as multilevel learning, supporting the view that a distinct type of physical theory, namely the theory of learning (<a href="#r35" rid="r35" class=" bibr popnode">35</a>, <a href="#r36" rid="r36" class=" bibr popnode">36</a>), is necessary to investigate the evolution of complex objects in the universe, of which evolution of life is a specific, even if highly remarkable form.</p></div><div id="s1" class="tsec sec"><h2 class="head no_bottom_margin" id="s1title" style="text-transform: uppercase;">1. Fundamental Principles of Evolution</h2><p class="p p-first">In this section, we attempt to formulate the minimal universal principles that define an observable universe, in which evolution is possible and perhaps, inevitable. Our analysis started from the major features of biological evolution discussed in the next section and proceeded toward the general principles. However, we begin the discussion with the latter for the sake of transparency and generality.</p><p>What are the requirements for a universe to be observable? The possibility to make meaningful observations implies a degree of order and complexity in the observed universe emerging from evolutionary processes, and such evolvability itself seems to be predicated on several fundamental principles. It has to be emphasized that &#x0201c;observation&#x0201d; and &#x0201c;learning&#x0201d; here by no means imply &#x0201c;mind&#x0201d; or &#x0201c;consciousness&#x0201d; but a far more basic requirement. To learn and survive in an environment, a system (or observer) must predict, with some minimal but sufficient degree of accuracy, the response of that environment to various actions and to be able to choose such actions that are compatible with the observer&#x02019;s continued existence in that environment. In this sense, any life-form is an observer, and so are even inanimate systems endowed with the ability of feedback reaction. In this most general sense, observation is a prerequisite for evolution. We first formulate the basic principles underlying observability and evolvability and then, give the pertinent comments and explanations.</p><ul class="simple" style="list-style-type:none"><li><div>P1. Loss function. In any evolving system, there exists a loss function of time-dependent variables that is minimized during evolution.</div></li><li><div>P2. Hierarchy of scales. Evolving systems encompass multiple dynamical variables that change on different temporal scales (with different characteristic frequencies).</div></li><li><div>P3. Frequency gaps. Dynamical variables are split among distinct levels of organization separated by sufficiently wide frequency gaps.</div></li><li><div>P4. Renormalizability. Across the entire range of organization of evolving systems, a statistical description of faster-changing (higher-frequency) variables is feasible through the slower-changing (lower-frequency) variables.</div></li><li><div>P5. Extension. Evolving systems have the capacity to recruit additional variables that can be utilized to sustain the system and the ability to exclude variables that could destabilize the system.</div></li><li><div>P6. Replication. In evolving systems, replication and elimination of the corresponding information-processing units (IPUs) can take place on every level of organization.</div></li><li><div>P7. Information flow. In evolving systems, slower-changing levels absorb information from faster-changing levels during learning and pass information down to the faster levels for prediction of the state of the environment and the system itself.</div></li></ul><p></p><p>The first principle (P1) is of special importance as the starting point for a formal description of evolution as a learning process. The very existence of a loss function implies that the dynamical system of the universe or simpler, the universe itself is a learning (evolving) system (<a href="#r36" rid="r36" class=" bibr popnode">36</a>). Effectively, here we assume that stability or survival of any subsystem of the universe is equivalent to solving an optimization or learning problem in the mathematical sense and that there is always something to learn. Crucially, for solving complex optimization problems dependent on many variables, the best and in fact, the only efficient method is selection implemented in various stochastic algorithms (Markov Chain Monte Carlo, stochastic gradient descent, genetic algorithms, and more). All evolution can be perceived as an implementation of a stochastic learning algorithm as well. Put another way, learning is optimization by trial and error, and so is evolution.</p><p>The remaining principles P2 to P7 provide sufficient conditions for observers of our type (that is, complex life-forms) to evolve within a learning system. In particular, P2, P3, and P4 comprise the necessary conditions for observability of a universe by any observer, whereas P5, P6, and P7 represent the defining conditions for the origin of life of our type (hereafter, we omit the qualification for brevity). More precisely, P2 and P3 provide for the possibility of at least a simple form of learning of the environment (fast-changing variables) by an observer (slow-changing variables) and hence, the emergence of complex organization of the slow-changing variables. P4 corresponds to the physical concept of renormalizability, or renormalization group (<a href="#r39" rid="r39" class=" bibr popnode">39</a>, <a href="#r40" rid="r40" class=" bibr popnode">40</a>), whereby the same macroscopic equations, albeit with different parameters, govern processes at different levels or scales, thus limiting the number of relevant variables, constraining the complexity, and allowing for a coarse-grained description. This principle ensures a renormalizable universe capable of evolution and amenable to observation. Together, P2 to P4 define a universe, in which partial or approximate knowledge of the environment (in other words, coarse graining) is both attainable and useful for the survival of evolving systems (observers). In a universe where P4 does not apply (that is, one with nonrenormalizable physical laws), what happens at the macroscopic level will critically depend on the details of the processes at the microlevel. In a universe where P2 and P3 do not apply, the separation of the micro- and macrolevels itself would not be apparent. In such a universe, it would be impossible to survive without first discovering fundamental physical laws, whereas living organisms on our planet have evolved for billions of years before starting to study quantum physics.</p><p class="p p-last">Principles P5, P6, and P7 endow evolving systems with the access to more advanced algorithms for learning and predicting the environment, paving the way for the evolution of complex systems, including eventually, life. These principles jointly underlie the emergence of the crucial phenomenon of selection (<a href="#r44" rid="r44" class=" bibr popnode">44</a>, <a href="#r45" rid="r45" class=" bibr popnode">45</a>). In its simplest form, selection is for stability and persistence of evolving, learning systems (<a href="#r46" rid="r46" class=" bibr popnode">46</a>). Learning and survival are tightly linked because survival is predicated on the system&#x02019;s ability to extract information from the environment, and this ability depends on the stability of the system on timescales required for learning. Roughly, a system cannot survive in a world where the properties of the environment change faster than the evolving system can learn them. According to P5, evolving systems consume resources (such as food), which themselves could be produced by other evolving systems, to be utilized as building blocks and energy sources, which are required for learning. This principle embodies Schroedinger&#x02019;s vision that &#x0201c;organisms feed on negentropy&#x0201d; (<a href="#r34" rid="r34" class=" bibr popnode">34</a>). Under P6, replication of the carriers of slowly changing variables becomes the basis of long-term persistence and memory in evolving systems. This principle can be viewed as a learning algorithm built on P3, whereby the timescales characteristic of an individual organism and of consecutive generations are separated. This principle excludes from consideration certain imaginary forms of life: for example, Stanislav Lem&#x02019;s famous Solaris (<a href="#r47" rid="r47" class=" bibr popnode">47</a>). Finally, P7 describes how information flows between different levels in the multilevel learning, giving rise to a generalized Central Dogma of molecular biology, which is discussed in <em>Generalized Central Dogma of Molecular Biology</em>.</p></div><div id="s2" class="tsec sec"><h2 class="head no_bottom_margin" id="s2title" style="text-transform: uppercase;">2. Fundamental Evolutionary Phenomena</h2><p class="p p-first">In this section, we link the fundamental principles of evolution P1 to P7 formulated above to the basic phenomenological features of life (E1 to E10) and seek equivalencies in the theory of learning. The list below is organized by first formulating a biological feature, and then, it is organized by 1) tracing the connections to the fundamental principles and 2) adding more general comments.</p><div id="s3" class="sec"><p></p><h3 id="s3title" class="inline">E1. IPUs. </h3><p class="p p-first-last">Discrete IPUs (that is, self- vs. nonself-differentiation and discrimination) exist at all levels of organization. All biological systems at all levels of organization, such as genes, cells, organisms, populations, and so on up to the level of the entire biosphere, possess some degree of self-coherence that separates them, first and foremost, from the environment at large and from other similar-level IPUs.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>The existence of IPUs is predicated on the fundamental principles P1 to P4. The wide range of temporal scales (P2) in dynamical systems and gaps between the scales (P3) naturally enable the separation of slower- and faster-changing components. In particular, renormalizability (P4) applies to the hierarchy of IPUs. The statistical predictability of the higher frequencies allows the IPUs to decrease the loss function of the lower frequencies, despite the much slower reaction times.</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>Separation of (relatively) slow-changing prebiological IPUs from the (typically) fast-changing environment kicked off the most primitive form of prebiological selection: selection for stability and persistence (survivor bias). More stable, slower-changing IPUs win in the competition and accumulate over time, increasing the separation along the temporal axis as the boundary between the IPUs and the environment grows sharper. Additional key phenomena, such as utilization of available environmental resources (P5) and the stimulus&#x02013;response mode of information exchange (P7), stem from the flow of matter and information across this boundary and the ensuing separation of internal and external physicochemical processes. Increasing self- vs. nonself-differentiation, combined with replication of the carriers of slow-changing variables (P6), sets the stage for competition between evolving entities and for the onset of the ultimate evolutionary phenomenon, natural selection (E6). </div></li></ul><p></p></div><div id="s4" class="sec"><p></p><h3 id="s4title" class="inline">E2. Frustration. </h3><p class="p p-first">All complex, dynamical systems face multidimensional and multiscale optimization problems, which generically lead to frustration resulting from conflicting objectives at different scales. This is a key, intrinsic feature of all such systems and a major force driving the advent of increasing multilevel complexity (<a href="#r12" rid="r12" class=" bibr popnode">12</a>). Frustration is an extremely general physical phenomenon that is by no account limited to biology but rather, occurs already in much simpler physical systems, such as spin and structural glasses, the behavior of which is determined by competing interactions so that a degree of complexity is attained (<a href="#r31" rid="r31" class=" bibr popnode">31</a>, <a href="#r32" rid="r32" class=" bibr popnode">32</a>).</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>The multiscale organization of the universe (P2) provides the physical foundation for the ubiquity of frustrated states that typically arises whenever there is a conflict (trade-off) between short- and long-range optimization problems. Frustrated interactions yield multiwell potential landscapes, in which no single state is substantially fitter than numerous other local optima. Multiparameter and multiscale optimization of the loss function on such a landscape involves nonergodic (history-dependent) dynamics, which is characteristic of complex systems.</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>IPUs face conflicting interactions starting from the most primitive prebiological state (<a href="#r12" rid="r12" class=" bibr popnode">12</a>). Indeed, the separation of any system from the environment immediately results in the conflict of permeability; a stronger separation enhances the self- vs. nonself-differentiation and thus, increases the stability of the system, but it compromises information and matter exchange with the environment, limiting the potential for growth. In biology, virtually all aspects of the organismal architecture and operation are subject to such frustrations or trade-offs: the conflict between the fidelity and speed of information transmission at all levels, between specialization and generalism, between the individual- and population-level benefits, and more. The ubiquity of frustrations and the fundamental impossibility of their resolution in a universally optimal manner are perpetual drivers of evolution and give rise to evolutionary transitions, attaining otherwise unreachable levels of complexity.</div></li></ul><p></p><p class="p p-last">There are two distinct types of frustrations, spatial and temporal. Spatial frustration is similar to the frustration that is commonly analyzed in condensed matter systems, such as spin glasses (<a href="#r31" rid="r31" class=" bibr popnode">31</a>, <a href="#r32" rid="r32" class=" bibr popnode">32</a>). In this case, the spatially local and nonlocal interacting terms have opposite signs so that the equilibrium state is determined by the balance between the terms. In neural networks, a neuron (like a single spin) might have a local objective (such as binary classification of incoming signals) but is also a part of a neural network (like a spin network), which has its own global objective (such as predicting its boundary conditions). For a particular neuron, optimization of the local objective can conflict with the global objective, causing spatial frustration. Temporal frustration emerges because in the context of multilevel learning, the same neuron becomes a part of higher-level IPUs that operate at different temporal scales (frequencies). Then, the optimal state of the neuron with respect to an IPU operating at a given timescale can differ from the optimal state of the same neuron with respect to another IPU operating at a different timescale (<a href="#r36" rid="r36" class=" bibr popnode">36</a>). Similarly to the spatial frustrations, temporal frustrations cannot be completely resolved, but an optimal balance between different spatial and temporal scales is achievable and represents a local equilibrium of the learning system.</p></div><div id="s5" class="sec"><p></p><h3 id="s5title" class="inline">E3. Multilevel Hierarchy. </h3><p class="p p-first-last">The hierarchy of multiple levels of organization is an intrinsic, essential feature of evolving biological systems in terms of both the structure of these systems (genes, genomes, cells, organisms, kin groups, populations, species, communities, and more) and the substrate the evolutionary forces act upon.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>Renormalizability of the universe (P4) implies that there is no inherently preferred level of organization, for which everything above and below would behave as a homogenous ensemble. Even if some levels of organization come into existence before others (for example, organisms before genes or unicellular organisms before multicellular ones), the other levels will necessarily emerge and consolidate subsequently.</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>The hierarchy of the structural organization of biological systems was apparent to scholars from the earliest days of science. However, MLS was and remains a controversial subject in evolutionary biology (<a href="#r23" rid="r23" class=" bibr popnode">23</a>, <a href="#r26" rid="r26" class=" bibr popnode">26</a>, <a href="#r27" rid="r27" class=" bibr popnode">27</a>). Intuitively and as implied by the Price equation (<a href="#r48" rid="r48" class=" bibr popnode">48</a>), MLS should emerge in all evolving systems as long as the higher-level agency of selection possesses a sufficient degree of self- vs. nonself-differentiation. In particular, if organisms of a given species form populations that are sufficiently distinct genetically and interact competitively, population-level selection will ensue. Evolution of biological systems is driven by conflicting interactions (E2) that tend to lead to ever-increasing complexity (<a href="#r12" rid="r12" class=" bibr popnode">12</a>). This trend further feeds the propensity of these systems to form new levels of organization and is associated with evolutionary transitions that involve the advent of new units of selection at multiple levels of complexity. Thus, E3 can be considered a major consequence of E2.</div></li></ul><p></p></div><div id="s6" class="sec"><p></p><h3 id="s6title" class="inline">E4. Near Optimality. </h3><p class="p p-first-last">Stochastic optimization or the use of stochastic optimization algorithms is the only feasible approach to complex optimization, but it guarantees neither finding the globally optimal solution nor retention of the optimal configuration when and if it is found. Rather, stochastic optimization tends to rapidly find local optima and keeps the system in their vicinity, sustaining the value of the loss function at a near-optimal level.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>According to P1, the dynamics of a learning (that is, self-optimizing) system is defined by a loss function (<a href="#r35" rid="r35" class=" bibr popnode">35</a>, <a href="#r36" rid="r36" class=" bibr popnode">36</a>). When there is a steep gradient in the loss function, a system undergoing stochastic optimization rapidly descends in the right direction. However, because of frustrations that inevitably arise from interactions in a complex system, actual local peaks on the landscape are rarely reached, and the global peak is effectively unreachable. Learning systems tend to get stalled near local saddle points where changes along most of the dimensions either lead &#x0201c;up&#x0201d; or are &#x0201c;flat&#x0201d; in terms of the loss function, with only a small minority of the available moves decreasing the loss function (<a href="#r49" rid="r49" class=" bibr popnode">49</a>).</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>The extant biological systems (cells, multicellular organisms, and higher-level entities, such as populations and communities) are products of about 4 billion y of the evolution of life, and as such, they are highly, albeit not completely, optimized. As a consequence, the typical distribution of the effects of heritable changes in biological evolution comprises numerous deleterious changes, comparatively rare beneficial changes and common neutral changes, and those with fitness effects below the noise level (<a href="#r50" rid="r50" class=" bibr popnode">50</a>). The preponderance of neutral and slightly deleterious changes provides for evolution by genetic drift whereby a population moves on the same level or even slightly downward on the fitness landscape, potentially reaching another region of the landscape where beneficial mutations are available (<a href="#r51" rid="r51" class=" bibr popnode">51</a>, <a href="#r52" rid="r52" class=" bibr popnode">52</a>).</div></li></ul><p></p></div><div id="s7" class="sec"><p></p><h3 id="s7title" class="inline">E5. Diversity of Near-Optimal Solutions. </h3><p class="p p-first-last">Solutions on the loss function landscapes that arise in complex optimization problems span numerous local peaks of comparable heights.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>The existence of multiple peaks of comparable heights in the loss function landscapes is a fundamental physical property of frustrated systems (E2), whereas the pervasiveness of frustration itself is a consequence of the multiscale and multilevel organization of the universe (P2). Frustrated dynamical systems are nonergodic, which from the biological perspective, means that, once separated, evolutionary trajectories diverge rather than converge. Because most of these trajectories traverse parts of the genotype space with comparable fitness values, competition rarely results in complete dominance of one lineage over the others but rather, generates rich diversity.</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>In terms of evolutionary biology, fitness landscapes are rugged, with multiple adaptive peaks of comparable fitness (<a href="#r53" rid="r53" class=" bibr popnode">53</a>, <a href="#r54" rid="r54" class=" bibr popnode">54</a>), and a salient trend during evolution is the spread of life-forms across multiple peaks as opposed to concentrating on one or few. Evolution pushes evolving organisms to explore and occupy all available niches and try all possible strategies. In the context of machine learning, identical neural networks can start from the same initial state but for example, under the stochastic gradient descent algorithm, would generically evolve toward different local minima. Thus, the diversity of solutions is a generic property of learning systems. More technically, the diversification is due to the entropy production through the dynamics of the neutral trainable variables (see the next section).</div></li></ul><p></p></div><div id="s8" class="sec"><p></p><h3 id="s8title" class="inline">E6. Separation of Phenotype from Genotype. </h3><p class="p p-first-last">This quintessential feature of life embodies two distinct (albeit inseparable in known organisms) symmetry-breaking phenomena: 1) separation between dedicated digital information storage media (stable, rarely updatable, tending to distributions with discrete values) and mostly analog processing devices and 2) asymmetry of the information flow within the IPUs whereby the genotype provides &#x0201c;instructions&#x0201d; for the phenotype, whereas the phenotype largely loses the ability to update the genotype directly. The separation between the information storage and processing subsystems is a prerequisite for efficient evolution that probably emerged early on the path from prebiotic entities to the emergence of life.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>The separation between phenotype and genotype extends the scale separation on the intra-IPU level as follows from the fundamental principles P1 to P4. Intermediate-frequency components of an IPU (phenotype) buffer the slowest components from direct interaction with the environment (the highest-frequency variables), further increasing the stability of the slowest components and making them suitable for long-term information storage. As the temporal scales separate further, the interactions between them change. Asymmetric information flow (P7) stabilizes the system, enabling long-term preservation of information (genotype) while retaining the reactive flexibility of the faster-changing components (phenotype).</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>The emergence of the separation between phenotype and genotype is a crucial event in prebiotic evolution. This separation is prominent in all known as well as hypothetical life-forms. Even when the phenotype and genotype roles are fulfilled by chemically identical molecules, as in the RNA world scenario of primordial evolution (<a href="#r55" rid="r55" class=" bibr popnode">55</a>, <a href="#r56" rid="r56" class=" bibr popnode">56</a>), their roles as effectors and information storage devices are sharply distinct. In biological terms, the split is between replicators (that is, the digital information carriers [genomes]) and reproducers (<a href="#r57" rid="r57" class=" bibr popnode">57</a><a href="#r58" rid="r58" class=" bibr popnode"></a>&#x02013;<a href="#r59" rid="r59" class=" bibr popnode">59</a>), the analog devices (cells, organisms) that host the replicators, supply them with building blocks (P5), and themselves reproduce (P6) under the replicators&#x02019; instruction (P7). Although the genotype/phenotype separation is a major staple of life, it is in itself insufficient to qualify an IPU as a life-form (computers and record players, in which the separation between information storage and operational parts is prominent and essential, clearly are not life, even though invented by advanced organisms). The asymmetry of information flow between genotype and phenotype (P7) is the most general form of the phenomenon known as the Central Dogma of molecular biology: the unidirectional flow of information from nucleic acids to proteins as originally formulated by Crick (<a href="#r60" rid="r60" class=" bibr popnode">60</a>). This asymmetry is also prominent in other information-processing systems, in particular computers. Indeed, von Neumann architecture computers have inherently distinct memory and processing units, with the instruction flow from the former to the latter (<a href="#r61" rid="r61" class=" bibr popnode">61</a>, <a href="#r62" rid="r62" class=" bibr popnode">62</a>). It appears that any advanced information-processing systems are endowed with this property.</div></li></ul><p></p></div><div id="s9" class="sec"><p></p><h3 id="s9title" class="inline">E7. Replication. </h3><p class="p p-first-last">Emergence of long-term digital storage devices, that is genomes consisting of RNA or DNA (E6) provides for long-term information preservation, facilitates adaptive reactions to changes in the environment, and promotes the stability of IPUs to the point where (at least in chemical systems) it is limited by the energy of the chemical bonds rather than the energy of thermal fluctuations. Obviously, however, as long as this information is confined to a single IPU, it will disappear with the inevitable eventual destruction of that IPU. Should this be the case, other IPUs of similar architecture would need to accumulate a comparable amount of information from scratch to reach the same level of stability. Thus, copying and sharing information are essential for long-term (effectively, indefinite) persistence of IPUs.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>The fundamental principle P6 postulates the existence of mechanisms for information copying and elimination. If genomic information can be replicated, even most primitive sharing mechanisms (such as physical splitting of an IPU under forces of surface tension) would result (even if not reliably) in the emergence of distinct IPUs preloaded with information that was amassed by their progenitor(s). This process short circuits learning and allows the information to accumulate at timescales far exceeding the characteristic lifetimes of individual IPUs.</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>Information copying and sharing are beneficial only if the fidelity exceeds a certain threshold, sometimes called Eigen limit in evolutionary biology (<a href="#r5" rid="r5" class=" bibr popnode">5</a><a href="#r6" rid="r6" class=" bibr popnode"></a>&#x02013;<a href="#r7" rid="r7" class=" bibr popnode">7</a>). Nevertheless, in primitive prebiotic systems, the required fidelity level could have been quite low (<a href="#r63" rid="r63" class=" bibr popnode">63</a>). For instance, even a biased chemical composition of a hydrophobic droplet could enhance the stability of the descendant droplets and thus, endow them with an advantage in the selection for persistence. However, once relatively sophisticated mechanisms of information copying and sharing emerge or more precisely, when replicators become information storage devices, the overall stability of the system can increase by orders of magnitude. To wit and astonishingly, the only biosphere known to us represents an unbroken chain of genetic information transmission that spans about 4 billion y, commensurate with the stellar evolution scale.</div></li></ul><p></p></div><div id="s10" class="sec"><p></p><h3 id="s10title" class="inline">E8. Natural Selection. </h3><p class="p p-first-last">Evolution by natural selection (Darwinian evolution) arises from the combination of all the principles and phenomena described above. The necessary and sufficient conditions for Darwinian evolution to operate are 1) the existence of IPUs that are distinct from the environment and from each other (E1), 2) the dependence of the stability of an IPU on the information it contains (that is, the phenotype&#x02013;genotype feedback; E6), and 3) the ability of IPUs to make copies of embedded information and share it with other IPUs (E7). When these three conditions are met, the relative frequencies of the more stable IPUs will increase with time via attrition of the less stable ones (survival of the fittest) and transfer of information among IPUs, both vertically (to progeny) and horizontally. This process engenders the key feature of Darwinian evolution, differential reproduction of genotypes, based on the feedback from the environment transmitted through the phenotype.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>All seven fundamental principles of life-compatible universes (P1 to P7) are involved in enabling evolution by natural selection. The very existence of units, on which selection can operate, hinges on self- vs. nonself-discrimination of prebiotic IPUs (E1) and the emergence of shareable information storage (E6 and E7). The crucial step to biology is the emergence of the link between the loss function (P1), on the one hand, and the existence of the IPUs (P2, P3, P4, and E1), on the other hand. Consumption of (limited) external resources (P5) entails competition between IPUs that share the same environment and turns mere shifts of the relative frequencies into true &#x0201c;survival of the fittest.&#x0201d; The ability of the IPUs to replicate (P6) and expand their memory storage (genotype; P7, E6, and E7) provides them with access to hitherto unavailable degrees of freedom, making evolution an open-ended process rather than a quick, limited search for a local optimum.</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>Evolution by natural selection is the central tenet of evolutionary biology and a key part of the NASA definition of life. An important note on definitions is due. We already referred to selection when discussing prebiotic evolution (E1); however, the term &#x0201c;natural (Darwinian) selection&#x0201d; is here reserved for the efficient form of selection that emerges with the replication of dedicated information storage devices (P6 and E6). Differential reproduction, whereby the environment provides feedback on the fitness of genotypes while acting on phenotypes, turns into Darwinian survival of the fittest in the presence of competition. When IPUs depend on environmental resources, such competition inevitably arises, except in the unrealistic case of unlimited supply (<a href="#r44" rid="r44" class=" bibr popnode">44</a>). With the onset of Darwinian evolution, the system can be considered to cross the threshold from prelife to life (<a href="#r64" rid="r64" class=" bibr popnode">64</a>, <a href="#r65" rid="r65" class=" bibr popnode">65</a>). The evolutionary process is naturally represented by movement of an evolving IPU in a genotype space, where proximity is defined by similarity between distinct genotypes and transitions correspond to elementary evolutionary events: that is, mutations in the most general sense (<a href="#r66" rid="r66" class=" bibr popnode">66</a>). For any given environment, fitness&#x02014;that is, a measure of the ability of a genotype to produce viable offspring&#x02014;can be defined for each point in the genotype space, forming a multidimensional fitness landscape (<a href="#r53" rid="r53" class=" bibr popnode">53</a>, <a href="#r54" rid="r54" class=" bibr popnode">54</a>). Selection creates a bias for preferential fixation of mutations that increase fitness, even if the mutations themselves occur completely randomly.</div></li></ul><p></p></div><div id="s11" class="sec"><p></p><h3 id="s11title" class="inline">E9. Parasitism. </h3><p class="p p-first-last">Parasites and host&#x02013;parasite coevolution are ubiquitous across biological systems at multiple levels of organization and are both intrinsic to and indispensable for the evolution of life.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>Due to the flexibility of life-compatible systems (P5 and P6) and the symmetry breaking in the information flow (P7) combined with the inherent tendency of life to diversify (E5), parts of the system inevitably settle on a parasitic state: that is, scavenging information and matter from the host without making a positive contribution to its fitness.</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>From the biological perspective, parasites evolve to minimize their direct interface with the environment and conversely, maximize their interaction with the host; in other words, the host replaces most of the environment for the parasite. Parasites inevitably emerge and persist in biological systems because of two reasons. 1) The parasitic state is reachable via an entropy-increasing step and therefore, is highly probable (<a href="#r16" rid="r16" class=" bibr popnode">16</a>), and 2) highly efficient antiparasite immunity is costly (<a href="#r67" rid="r67" class=" bibr popnode">67</a>). The cost of immunity reflects another universal trade-off analogous to the trade-off between information transfer fidelity and energy expenditure; in both cases, an infinite amount of energy is required to reach a zero error rate or a parasite-free state. From a complementary standpoint, parasites inevitably evolve as cheaters in the game of life that exploit the host as a resource, without expending energy on resource production. Short-term, parasites reduce the host fitness by both direct drain on its resources and various indirect effects, including the cost of defense. However, in a longer-term perspective, parasites make up a reservoir for recruitment of new functions (especially, but far from exclusively, for defense) by the hosts (<a href="#r14" rid="r14" class=" bibr popnode">14</a>, <a href="#r15" rid="r15" class=" bibr popnode">15</a>). The host&#x02013;parasite relationship can evolve toward transition to a mutually beneficial, symbiotic lifestyle that can further progress to mutualism and in some cases, complete integration as exemplified by the origin of essential endosymbiotic organelles in eukaryotes, mitochondria, and chloroplasts (<a href="#r68" rid="r68" class=" bibr popnode">68</a>, <a href="#r69" rid="r69" class=" bibr popnode">69</a>). Parasites emerge at similar levels of biological organization (organisms parasitizing other organisms) or across levels (genetic elements parasitizing organismal genomes or cell clones parasitizing multicellular organisms).</div></li></ul><p></p></div><div id="s12" class="sec sec-last"><p></p><h3 id="s12title" class="inline">E10. Programmed Death. </h3><p class="p p-first">Programmed (to various degrees) death is an intrinsic feature of life.</p><ul class="simple" style="list-style-type:none"><li class="a_label"><div class="inline_block a_label">1)</div><div>Replication and elimination of IPUs (P6) and utilization of additional degrees of freedom (P5) form the foundation for the phenomenon of programmed death. At some levels of organization (for example, intragenomic), the ability to add and eliminate units (such as genes) for the benefit of the higher-level systems (such as organisms) provides an obvious path of optimization. Elimination of units could be, in principle, completely random, but selection (E8) generates a sufficiently strong feedback to facilitate and structure the loss process (for example, purging low-fitness genes via homologous recombination or altruistic suicide of infected or otherwise impaired cells). The same forces operate at least at the cell level and conceivably, at all levels of organization and selection (P4). In particular, if population-level or kin-level selection is sufficiently strong, mechanisms for altruistic death of individual organisms apparently can be fixed in evolution (<a href="#r70" rid="r70" class=" bibr popnode">70</a>, <a href="#r71" rid="r71" class=" bibr popnode">71</a>).</div></li><li class="a_label"><div class="inline_block a_label">2)</div><div>Programmed death is a prominent case of minimization of the higher-level (for example, organismal) loss function at the cost of increasing the lower-level loss function (such as that of individual cells). Although (tightly controlled) programmed cell death was originally discovered in multicellular organisms and has been thought to be limited to these complex life-forms, altruistic cell suicide now appears to be a universal biological phenomenon (<a href="#r71" rid="r71" class=" bibr popnode">71</a><a href="#r72" rid="r72" class=" bibr popnode"></a>&#x02013;<a href="#r73" rid="r73" class=" bibr popnode">73</a>). </div></li></ul><p></p><p class="p p-last">To conclude this section, which we titled &#x0201c;fundamental evolutionary phenomena,&#x0201d; deliberately omitting &#x0201c;biological,&#x0201d; it seems important to note that phenomena E1 to E7 are generic, applying to all learning systems, including purely physical and prebiotic ones. However, the onset of natural selection (E8) marks the origin of life, so that the phenomena E8 to E10 belong in the realm of biology.</p></div></div><div id="s13" class="tsec sec"><h2 class="head no_bottom_margin" id="s13title" style="text-transform: uppercase;">3. Optimization and Scale Separation in Evolving Systems</h2><p class="p p-first">In the previous sections, we formulated the seven fundamental principles of evolution P1 to P7 and then, argued that the key evolutionary phenomena E1 to E10 can be interpreted and analyzed in the context of these principles and apparently, derived from the latter. The next step is to formulate a mathematical framework that would be consistent with the fundamental principles and thus, would allow us to model evolutionary phenomena analytically or numerically. For concreteness, the proposed framework is based on a mathematical model of artificial neural networks (<a href="#r74" rid="r74" class=" bibr popnode">74</a>, <a href="#r75" rid="r75" class=" bibr popnode">75</a>), but we first outline a general optimization approach in a form suitable for modeling biological evolution.</p><p>We are interested in the broadest class of optimization problems, where the loss (or cost) function <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i1" display="inline" overflow="scroll"><mrow><mi>H</mi><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow></mrow></math> is minimized with respect to some trainable variables,</p><div class="disp-formula" id="eqn1"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d1" display="block" overflow="scroll"><mrow><mi mathvariant="bold">q</mi><mo>=</mo><mrow><mo stretchy="true">(</mo><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">c</mi><mo stretchy="false">)</mo></mrow></msup><mo>,</mo><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">a</mi><mo stretchy="false">)</mo></mrow></msup><mo>,</mo><mo>&#x000a0;</mo><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo stretchy="true">)</mo></mrow><mo>,</mo></mrow></math></div><div class="l" style="width:3.50em">[3.1]</div></div><p>for a given training set of nontrainable variables,</p><div class="disp-formula" id="eqn2"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d2" display="block" overflow="scroll"><mrow><mi mathvariant="bold">x</mi><mo>=</mo><mrow><mo stretchy="true">(</mo><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">o</mi><mo stretchy="false">)</mo></mrow></msup><mo>,</mo><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo stretchy="true">)</mo></mrow><mo>.</mo></mrow></math></div><div class="l" style="width:3.50em">[3.2]</div></div><p></p><p>Near a local minimum, the first derivatives of the average loss function with respect to trainable variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i2" display="inline" overflow="scroll"><mi mathvariant="bold">q</mi></math> are small, and the depth of the minimum usually depends on the second derivatives. In particular, the second derivative can be large for the effectively constant degrees of freedom, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i3" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">c</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>; small for adaptable degrees of freedom, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i4" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">a</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>; or near zero for symmetries or neutral directions <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i5" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>. The separation of the neutral directions <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i6" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> into a special class of variables simply means that some of the trainable variables can be changed without affecting the learning outcome: that is, the value of the loss function. Put another way, neutral changes are always possible. The neutral directions <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i7" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> are the fastest changing among the trainable variables because fluctuations resulting in their change are, in general, fully stochastic. On the other end of the spectrum of variables, even minor changes to the effectively constant variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i8" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">c</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> compromise the entire learning (evolution) process: that is, result in a substantial increase of the loss function value; these variables correspond to deep minima of the loss function. When the basin of attraction of a minimum is deep and narrow, the system stays in its bottom for a long time, and then, to describe such a state, it is sufficient to use discrete information (that is, to indicate that the system stays in a given minimum) rather than to list all specific values of the coordinates in a multidimensional space.</p><p>In a generic optimization problem, the dynamics of both trainable and nontrainable variables involves a broad distribution of characteristic timescales <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i9" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math>, and switching between scales is equivalent to switching between different frequencies or in the context of biological evolution, between different levels of organization. For any fixed <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i10" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math>, all variables can be partitioned into three classes depending on how fast they change with respect to the specified timescale:</p><ul class="simple" style="list-style-type:none"><li><div>fast-changing nontrainable variables that characterize an organism<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i11" display="inline" overflow="scroll"><mo>&#x000a0;</mo><mo>(</mo><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="true">(</mo><mi mathvariant="bold-italic">o</mi><mo stretchy="true">)</mo></mrow></msup><mo>)</mo></math> and its environment <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i12" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> and change on timescales <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i13" display="inline" overflow="scroll"><mo>&#x0226a;</mo><mi mathvariant="italic">&#x003c4;</mi></math>;</div></li><li><div>intermediate-changing adaptable variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i14" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">a</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> or neutral directions <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i15" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> that change on timescales ~<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i16" display="inline" overflow="scroll"><mi>&#x003c4;</mi></math>; and</div></li><li><div>slow-changing variables, which are the degrees of freedom <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i17" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">c</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> that have already been well trained and are effectively constant (at or near equilibrium), only changing on timescales <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i18" display="inline" overflow="scroll"><mo>&#x0226b;</mo><mi mathvariant="italic">&#x003c4;</mi></math>.</div></li></ul><p></p><p>As will become evident shortly, the separation of these three classes of variables and interactions between them are central to the evolution and selection on all levels of organization, resulting in pervasive multilevel learning and selection.</p><p class="p p-last">Depending on the considered timescale <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i19" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math> (or as a result of environmental changes), the same dynamical degree of freedom can be assigned to different classes of variables: that is, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i20" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">o</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i21" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i22" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">c</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i23" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">a</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>, or <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i24" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>. For example, on the shortest timescale, which corresponds to the lifetime of an individual organism (one generation), the adaptable variables are the phenotypic traits that quickly respond to environmental changes, whereas the slowest, near-constant variables are the genomic sequences (genotype) that change minimally if at all. On longer timescales, corresponding to thousands or millions of generations, fast-evolving portions of the genome become adaptable variables, whereas the conserved core of the genome remains in the near-constant class (<a href="#r50" rid="r50" class=" bibr popnode">50</a>). Analogously, the neutral directions correspond either to nonconsequential phenotypic changes or to neutral genomic mutations, depending on the timescale. It is well known that the overwhelming majority of the mutations are either deleterious and therefore, eliminated by purifying selection or (nearly) neutral and thus, can be either lost or fixed via drift (<a href="#r76" rid="r76" class=" bibr popnode">76</a>, <a href="#r77" rid="r77" class=" bibr popnode">77</a>). However, when the environment changes or under the influence of other mutations, some of the neutral mutations can become beneficial [a genetic phenomenon known as epistasis, which is pervasive in evolution (<a href="#r78" rid="r78" class=" bibr popnode">78</a>, <a href="#r79" rid="r79" class=" bibr popnode">79</a>)], and in their entirety, neutral mutations form the essential reservoir of variation available for adaptive evolution (<a href="#r80" rid="r80" class=" bibr popnode">80</a>). Even which variables are classified as nontrainable <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i25" display="inline" overflow="scroll"><mrow><mo stretchy="false">(</mo><mi mathvariant="bold">x</mi><mo stretchy="false">)</mo></mrow></math> depends on the timescale <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i26" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math>. For example, if a learning system was trained for a sufficiently long time, some of the trainable variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i27" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">a</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> or <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i28" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> might have already equilibrated and become nontrainable.</p></div><div id="s14" class="tsec sec"><h2 class="head no_bottom_margin" id="s14title" style="text-transform: uppercase;">4. The Neural Network Framework</h2><p class="p p-first">Now that we described an optimization problem that is suitable for modeling evolution of organisms (or populations of organisms), we can construct a mathematical framework to solve such optimization problems. For this purpose, we employ a mathematical theory of artificial neural networks (<a href="#r74" rid="r74" class=" bibr popnode">74</a>, <a href="#r75" rid="r75" class=" bibr popnode">75</a>), which is simple enough to perform calculations while being consistent with all of the fundamental principles (P1 to P7), and thus, it can be used for modeling evolutionary phenomena (E1 to E10). We first recall a general framework of the neural network theory.</p><p>Consider a learning system represented as a neural network, with the state vector described by trainable variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i29" display="inline" overflow="scroll"><mi mathvariant="bold">q</mi></math> (which describe a collective notation for weight matrix <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i30" display="inline" overflow="scroll"><mover accent="true"><mi>w</mi><mo>^</mo></mover></math> and bias vector <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i31" display="inline" overflow="scroll"><mi mathvariant="bold">b</mi></math>) and nontrainable variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i32" display="inline" overflow="scroll"><mi mathvariant="bold">x</mi></math> (which describe the current state vector of individual neurons). In the biological context, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i33" display="inline" overflow="scroll"><mi mathvariant="bold">x</mi></math> collectively represent the current state of the organism <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i34" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">o</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> and of its environment <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i35" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>, and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i36" display="inline" overflow="scroll"><mi mathvariant="bold">q</mi></math> determines how <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i37" display="inline" overflow="scroll"><mi mathvariant="bold">x</mi></math> changes with time, in particular, how the organism reacts to environmental challenges. The nontrainable variables are modeled as changing in discrete time steps</p><div class="disp-formula" id="eqn3"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d4" display="block" overflow="scroll"><mrow><msub><mtext>x</mtext><mi>i</mi></msub><mrow><mo stretchy="true">(</mo><mrow><mi>t</mi><mo>+</mo><mn>1</mn></mrow><mo stretchy="true">)</mo></mrow><mo>=</mo><msub><mi>f</mi><mi>i</mi></msub><mrow><mo stretchy="true">(</mo><mrow><munder><mstyle displaystyle="true"><mo>&#x02211;</mo></mstyle><mi>j</mi></munder><msub><mi>w</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><msub><mi>x</mi><mi>j</mi></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>+</mo><msub><mi>b</mi><mi>i</mi></msub></mrow><mo stretchy="true">)</mo></mrow><mo>,</mo></mrow></math></div><div class="l" style="width:3.50em">[4.1]</div></div><p>where <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i38" display="inline" overflow="scroll"><mrow><msub><mi>f</mi><mi>i</mi></msub><mo stretchy="false">(</mo><mi>y</mi><mo stretchy="false">)</mo></mrow></math>&#x02019;s are some nonlinear activation functions (for example, hyperbolic tangent or rectifier activation functions). The trainable variables are modeled as changing according to the gradient descent (or stochastic gradient descent) algorithm</p><div class="disp-formula" id="eqn4"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d5" display="block" overflow="scroll"><mrow><msub><mi>q</mi><mi>i</mi></msub><mrow><mo stretchy="true">(</mo><mrow><mi>t</mi><mo>+</mo><mn>1</mn></mrow><mo stretchy="true">)</mo></mrow><mo>=</mo><msub><mi>q</mi><mi>i</mi></msub><mrow><mo stretchy="true">(</mo><mi>t</mi><mo stretchy="true">)</mo></mrow><mo>&#x02212;</mo><mi mathvariant="italic">&#x003b3;</mi><mfrac><mrow><mo>&#x02202;</mo><mi>H</mi><mo stretchy="false">(</mo><mi mathvariant="bold">x</mi><mrow><mo stretchy="true">(</mo><mi>t</mi><mo stretchy="true">)</mo></mrow><mo>,</mo><mi mathvariant="bold">q</mi><mrow><mo stretchy="true">(</mo><mi>t</mi><mo stretchy="true">)</mo></mrow><mo stretchy="false">)</mo></mrow><mrow><mo>&#x02202;</mo><msub><mi>q</mi><mi>i</mi></msub></mrow></mfrac><mo>,</mo></mrow></math></div><div class="l" style="width:3.50em">[4.2]</div></div><p></p><p>where <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i39" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003b3;</mi></math> is the learning rate parameter and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i40" display="inline" overflow="scroll"><mrow><mi>H</mi><mo stretchy="false">(</mo><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi><mo stretchy="false">)</mo></mrow></math> is a suitably defined loss function (<a href="#eqn3" rid="eqn3" class=" disp-formula">Eqs. <strong>4.3</strong></a> and <a href="#eqn6" rid="eqn6" class=" disp-formula"><strong>4.4</strong></a>). In other words, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i41" display="inline" overflow="scroll"><mi mathvariant="bold">q</mi></math> are &#x0201c;gross&#x0201d; or &#x0201c;main&#x0201d; variables, which determine the rules of dynamics, and the dynamics of all other variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i42" display="inline" overflow="scroll"><mi mathvariant="bold">x</mi></math> is governed by these rules, per <a href="#eqn3" rid="eqn3" class=" disp-formula">Eq. <strong>4.1</strong></a>. In the biological context, <a href="#eqn3" rid="eqn3" class=" disp-formula">Eq. <strong>4.1</strong></a> represents fast, often stochastic environmental changes and the corresponding fast reaction of organisms at the phenotype level, whereas [<a href="#eqn4" rid="eqn4" class=" disp-formula"><strong>4.2</strong></a>] reflects slower-learning dynamics of evolutionary adaptation via changes in the intermediate, adaptable variables: that is, the variable portion of the genome. The main learning objective is to adjust the trainable variables such that the average loss function is minimized subject to boundary conditions (also known as the training dataset), which in our case, is modeled as a time sequence of the environmental variables.</p><p>For example, on a single-generation timescale, the fast-changing variables represent the environment <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i43" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> and nontrainable variables associated with organisms <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i44" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">o</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math>, the intermediate-changing variables represent adaptive <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i45" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">a</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> and neutral <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i46" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> phenotype changes, and the slow-changing variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i47" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">c</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> represent the genotype (<a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2120037119/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S1</a>).</p><p>The temporal-scale separation in biology is readily apparent in all organisms. Indeed, consequential changes in the environment <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i48" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">e</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> often occur on the scale of milliseconds to seconds, triggering physical changes within organisms <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i49" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">o</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> at matching timescales. In response, individual organisms respond with phenotypic changes both adaptive <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i50" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">a</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> and neutral <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i51" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">n</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> on the scale of minutes to hours, exploiting their genetically encoded phenotypic plasticity. A paradigmatic example is induction of bacterial operons in response to a change in the chemical composition of the environment, such as the switch from glucose to galactose as the primary nutrient (<a href="#r81" rid="r81" class=" bibr popnode">81</a>, <a href="#r82" rid="r82" class=" bibr popnode">82</a>). In contrast, changes in the genome <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i52" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="false">(</mo><mi mathvariant="bold-italic">c</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math> take much longer. Mutations typically occur at rates of about 1 to 10 per genome replication cycle (<a href="#r83" rid="r83" class=" bibr popnode">83</a>), which for unicellular organisms, is the same as a generation comprising from about an hour to hundreds or even thousands of hours. However, fixation of mutations, which represents an evolutionarily stable change at the genome level, typically takes many generations and thus, always occurs orders of magnitude slower than phenotype changes. Accordingly, on this timescale, any changes in the genome represent the third layer in the network, the slowly changing variables.</p><p>To specify a microscopic loss function that would be appropriate for describing evolution and thus, give a specific form to the fundamental principle P1, we first note that adaptation to the environment is more efficient (that is, the loss function value is smaller) for a learning system, such as an organism, that can predict the state of its environment with a smaller error. Then, the relevant quantity is the so-called &#x0201c;boundary&#x0201d; loss function defined as the sum of squared errors,</p><div class="disp-formula" id="eqn5"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d6" display="block" overflow="scroll"><mrow><msub><mi>H</mi><mi>e</mi></msub><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow><mo>&#x02261;</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><munder><mstyle displaystyle="true"><mo>&#x02211;</mo></mstyle><mrow><mi>i</mi><mo>&#x02208;</mo><mi mathvariant="normal">E</mi></mrow></munder><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><msubsup><mi>x</mi><mi>i</mi><mrow><mo stretchy="false">(</mo><mi>e</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>&#x02212;</mo><msub><mi>f</mi><mi>i</mi></msub><mo stretchy="false">(</mo><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>o</mi><mo stretchy="true">)</mo></mrow></mrow></msup><mo>,</mo><mi mathvariant="bold">q</mi><mo stretchy="false">)</mo></mrow><mo stretchy="true">)</mo></mrow></mrow><mn>2</mn></msup><mo>,</mo></mrow></math></div><div class="l" style="width:3.50em">[4.3]</div></div><p>where the summation is taken only over the boundary (or environmental) nontrainable variables. It is helpful to think of the boundary loss function as the mismatch between the actual state of the environment and the state that would be predicted by the neural network if the environmental dynamics was switched off. In neuroscience, boundary loss is closely related to the surprise (or prediction error) associated with predictions of sensations, which depend on an internal model of the environment (<a href="#r84" rid="r84" class=" bibr popnode">84</a>). In machine learning, boundary loss functions are most often used in the context of supervised learning (<a href="#r35" rid="r35" class=" bibr popnode">35</a>), and in biological evolution, the &#x0201c;supervision&#x0201d; comes from the environment, which the evolving system, such as an organism or a population, is learning to predict.</p><p>Another possibility for a learning system is to search for the minimum of the &#x0201c;bulk&#x0201d; loss function, which is defined as the sum of squared errors over all neurons:</p><div class="disp-formula" id="eqn6"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d7" display="block" overflow="scroll"><mrow><mi>H</mi><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><munder><mstyle displaystyle="true"><mo>&#x02211;</mo></mstyle><mi>i</mi></munder><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><msub><mi>x</mi><mi>i</mi></msub><mo>&#x02212;</mo><msub><mi>f</mi><mi>i</mi></msub><mo stretchy="false">(</mo><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>o</mi><mo stretchy="true">)</mo></mrow></mrow></msup><mo>,</mo><mi mathvariant="bold">q</mi><mo stretchy="false">)</mo></mrow><mo stretchy="true">)</mo></mrow></mrow><mn>2</mn></msup><mo>.</mo></mrow></math></div><div class="l" style="width:3.50em">[4.4]</div></div><p></p><p>The bulk loss function assumes extra cost incurred by changing the states of organismal neurons, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i53" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>o</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math>: that is, rewarding stationary states. In the limit of a very large number of environmental neurons, the two loss functions are indistinguishable, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i54" display="inline" overflow="scroll"><mrow><mi>H</mi><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow><mo>&#x02248;</mo><msub><mi>H</mi><mi>e</mi></msub><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow></mrow></math>, but bulk loss is easier to handle mathematically (the details of boundary and bulk loss functions are addressed in ref. <a href="#r35" rid="r35" class=" bibr popnode">35</a>).</p><p>More generally, in addition to the &#x0201c;kinetic&#x0201d; term [<a href="#eqn6" rid="eqn6" class=" disp-formula"><strong>4.4</strong></a>], the loss function can include a &#x0201c;potential&#x0201d; term <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i55" display="inline" overflow="scroll"><mrow><mi>V</mi><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow></mrow></math>:</p><div class="disp-formula" id="eqn7"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d8" display="block" overflow="scroll"><mrow><mi>H</mi><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow><mo>=</mo><mfrac><mn>1</mn><mn>2</mn></mfrac><munder><mstyle displaystyle="true"><mo>&#x02211;</mo></mstyle><mi>i</mi></munder><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><msub><mi>x</mi><mi>i</mi></msub><mo>&#x02212;</mo><msub><mi>f</mi><mi>i</mi></msub><mo stretchy="false">(</mo><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>o</mi><mo stretchy="true">)</mo></mrow></mrow></msup><mo>,</mo><mi mathvariant="bold">q</mi><mo stretchy="false">)</mo></mrow><mo stretchy="true">)</mo></mrow></mrow><mn>2</mn></msup><mo>+</mo><mi>V</mi><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow><mo>.</mo></mrow></math></div><div class="l" style="width:3.50em">[4.5]</div></div><p></p><p>The kinetic term in [<a href="#eqn7" rid="eqn7" class=" disp-formula"><strong>4.5</strong></a>] reflects the ability of organisms <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i56" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>o</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> to predict the changes in the state of the given environment <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i57" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> over time, whereas the potential term reflects their compatibility with a given environment and hence, the capacity to choose among different environments.</p><p>In the context of biological evolution, Malthusian fitness <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i58" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c6;</mi></math> is defined as the expected reproductive success of a given genotype: that is, the rate of change of the prevalence of the given genotype in an evolving population (<a href="#r85" rid="r85" class=" bibr popnode">85</a>). However, in the context of the theory of learning, the loss function must be identified with additive fitness: that is,</p><div class="disp-formula" id="eqn8"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d9" display="block" overflow="scroll"><mrow><mi>H</mi><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow><mo>=</mo><mo>&#x02212;</mo><mi>T</mi><mi>log</mi><mo>&#x0200a;</mo><mi mathvariant="italic">&#x003c6;</mi><mrow><mo stretchy="true">(</mo><mrow><mi mathvariant="bold">x</mi><mo>,</mo><mi mathvariant="bold">q</mi></mrow><mo stretchy="true">)</mo></mrow><mo>.</mo></mrow></math></div><div class="l" style="width:3.50em">[4.6]</div></div><p></p><p>For a microscopic description of learning, the proportionality constant is unimportant, but as we argue in detail in the accompanying paper (<a href="#r86" rid="r86" class=" bibr popnode">86</a>), in the description of the evolutionary process from the point of view of thermodynamics, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i59" display="inline" overflow="scroll"><mi>T</mi></math> plays the role of &#x0201c;evolutionary temperature.&#x0201d;</p><p class="p p-last">Given a concrete mathematical model of neural networks, one might wonder if all fundamental principles of evolution (P1 to P7) can be derived from this model. Such derivation would comprise additional evidence supporting the claim that the entire universe can be adequately described as a neural network (<a href="#r36" rid="r36" class=" bibr popnode">36</a>). Clearly, the existence of a loss function (P1) follows automatically because learning of any neural network is always described relative to a specified loss function (<a href="#eqn3" rid="eqn3" class=" disp-formula">Eq. <strong>4.4</strong></a> or <a href="#eqn7" rid="eqn7" class=" disp-formula"><strong>4.5</strong></a>). The other six principles also seem to naturally emerge from the learning dynamics of neural networks. In particular, the hierarchy of scales (P2) and frequency gaps (P3) are generic consequences of the learning dynamics, whereby a system that involves a wide range of variables changing at different rates is attracted toward a self-organized critical state of slow-changing trainable variables (<a href="#r87" rid="r87" class=" bibr popnode">87</a>). Additional gaps between levels of organization are also expected to appear through phase transitions as becomes apparent in the thermodynamic description of evolution we develop in the accompanying paper (<a href="#r86" rid="r86" class=" bibr popnode">86</a>). Renormalizability (P4) is a direct consequence of the second law of learning (<a href="#r35" rid="r35" class=" bibr popnode">35</a>), according to which entropy of a system (and consequently, complexity of neural network or rank of its weight matrix) decreases with learning. This phenomenon was observed in neural network simulations (<a href="#r35" rid="r35" class=" bibr popnode">35</a>) and is the exact type of dynamics that can make the system renormalizable even if it started off as a highly entangled (large rank of weight matrix), nonrenormalizable neural network. The extension (P5) and replication (P6) principles simply indicate that additional variables can lead to either increase or decrease in the value of the loss function (<a href="#r35" rid="r35" class=" bibr popnode">35</a>). It is also important to note that in neural networks, an additional computational advantage (&#x0201c;quantum advantage&#x0201d;) can be achieved if the number of IPUs can vary (<a href="#r88" rid="r88" class=" bibr popnode">88</a>). Therefore, to achieve such an advantage, a system must learn how to replicate and eliminate its IPUs (P6). Finally, in <em>Generalized Central Dogma of Molecular Biology</em>, we illustrate how Fourier transform (or more generally, wavelet transform) of the environmental degrees of freedom can be used for learning the environment and how the inverse transform can be used for predicting it. Thus, to be able to predict the environment (and hence, to be competitive), any evolving system must learn the mechanism behind such asymmetric information flow (P7).</p></div><div id="s15" class="tsec sec"><h2 class="head no_bottom_margin" id="s15title" style="text-transform: uppercase;">5. Multilevel Learning</h2><p class="p p-first">In the previous sections, we argued that the learning process naturally divides all the dynamical variables into three distinct classes: fast-changing ones, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i60" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>o</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i61" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math>; intermediate-changing ones, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i62" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>a</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i63" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>n</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> (<math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i64" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>n</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> being faster than <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i65" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>a</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math>); and slow-changing ones, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i66" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>c</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> (<a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2120037119/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S1</a>). Evidently, this separation of variables depends on the timescale <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i67" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math> during which the system is observed, and variables migrate between classes when <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i68" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math> is increased or decreased (<a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2120037119/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2</a>). The longer the time, the more variables reach equilibrium and therefore, can be modeled as nontrainable and fast changing, <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i69" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math>, and the fewer variables remain slowly varying and can be modeled as effectively constant <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i70" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>c</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math>. In other words, many variables that are nearly constant at short timescales migrate to the intermediate class at longer timescales, whereas variables from the intermediate class migrate to the fast class.</p><p>In biological terms, if we consider learning dynamics on the timescale of one generation, then <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i71" display="inline" overflow="scroll"><mrow><mo>&#x000a0;</mo><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>a</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i72" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>n</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> represent phenotype variables, and <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i73" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>c</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math> represents genotype variables; however, on much longer timescales of multiple generations, the learning dynamics of populations (or communities) of organisms becomes relevant. On such timescales, the genotype variables acquire dynamics, with purifying and positive selection getting into play, whereas the phenotype variables progressively equilibrate. There is a clear connection between learning dynamics, including that in biological systems, and renormalizability of physical theories (P4). Indeed, from the point of view of an efficient learning algorithm, the parameters controlling learning dynamics, such as effective learning (or information-processing) rate <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i74" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003b3;</mi></math>, can vary from one timescale to another (for example, from individual organisms to populations or communities of organisms), but the general principles as well as specific dependencies captured in the equations above that govern the learning dynamics on different timescales remain the same. We refer to this universality of the learning process on different timescales and partitioning of the variables into temporal classes as multilevel learning.</p><p>More precisely, multilevel learning is a property of learning systems, which allows for the basic equations of learning, such as [<a href="#eqn6" rid="eqn6" class=" disp-formula"><strong>4.4</strong></a>], to remain the same on all levels of organization but for the parameters, which describe the dynamics such as <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i75" display="inline" overflow="scroll"><mi>&#x003b3;</mi><mo>(</mo><mi mathvariant="italic">&#x003c4;</mi><mo>)</mo></math>, to depend on the level or on the timescale <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i76" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math>. For example, if the effective learning (or information-processing) rate <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i77" display="inline" overflow="scroll"><mi>&#x003b3;</mi><mo>(</mo><mi mathvariant="italic">&#x003c4;</mi><mo>)</mo></math> decreases with timescale <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i78" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math>, then the local processing time, which depends on <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i79" display="inline" overflow="scroll"><mi>&#x003b3;</mi><mo>(</mo><mi mathvariant="italic">&#x003c4;</mi><mo>)</mo></math>, runs differently for different trainable variables: slower for slow-changing variables (or larger <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i80" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math>) and faster for fast-changing ones (or smaller <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i81" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math>). For such a system, the concept of global time (that is, the same time for all variables) becomes irrelevant and should be replaced with the proper or local time, which is defined for each scale <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i82" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math> separately:</p><div class="disp-formula" id="eqn9"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d10" display="block" overflow="scroll"><msub><mi>t</mi><mi mathvariant="italic">&#x003c4;</mi></msub><mo>&#x0221d;</mo><mi>&#x003b3;</mi><mo>(</mo><mi mathvariant="italic">&#x003c4;</mi><mo>)</mo><mi>t</mi><mo>.</mo></math></div><div class="l" style="width:3.50em">[5.1]</div></div><p></p><p class="p p-last">This effect closely resembles time dilation phenomena in physics, except that in special and general relativity, time dilation is linked with the possibility of movement between slow and fast clocks (or variables) (<a href="#r89" rid="r89" class=" bibr popnode">89</a>). To illustrate the role time dilation plays in biology, consider only two types of variables: slow changing and fast changing. Then, the slow variables should be able to &#x0201c;outsource&#x0201d; certain computational tasks to faster variables. Because the local clock for the fast-changing variables runs faster, the slow-changing variables can take advantage of the fast-changing ones to accelerate computation, which would be rewarded by evolution. The flow of information between slow-changing and fast-changing variables in the opposite direction is also beneficial because the fast-changing variables can use the slow-changing variables to store useful information for future retrieval: that is, the slow variables function as long-term memory. In the next section, we show that such cooperation between slow- and fast-changing variables, which is a concrete manifestation of principle P7, corresponds to a crucial biological phenomenon as the Central Dogma of molecular biology (<a href="#r60" rid="r60" class=" bibr popnode">60</a>).</p></div><div id="s16" class="tsec sec"><h2 class="head no_bottom_margin" id="s16title" style="text-transform: uppercase;">6. Generalized Central Dogma of Molecular Biology</h2><p class="p p-first">In terms of learning theory, the two directions of the asymmetric information flow (P7) represent learning the state of the environment and predicting the state of the environment from the results of learning. For learning, information is passed from faster variables to slower variables, and for predicting, information flows in the opposite direction from slower variables to the faster ones. A more formal analysis of the asymmetric information flows (or a generalized Central Dogma) can be carried out by forward propagation (from slow variables to fast variables) and back propagation (from fast variables to slow variables) of information within the framework of the mathematical model of neural networks developed in the previous sections (<a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2120037119/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S3</a>).</p><p>Consider nontrainable environmental variables that change continuously with time <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i83" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></mrow></msup><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math>, while the learning objective of an organism is to predict <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i84" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></mrow></msup><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math> at time <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i85" display="inline" overflow="scroll"><mi>t</mi><mo>&#x0003e;</mo><mi mathvariant="italic">&#x003c4;</mi></math> given that it was observed for time <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i86" display="inline" overflow="scroll"><mn>0</mn><mo>&#x0003c;</mo><mi>t</mi><mo>&#x0003c;</mo><mi mathvariant="italic">&#x003c4;</mi></math>. Thus, the organism has to extrapolate the function <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i87" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></mrow></msup><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math> for <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i88" display="inline" overflow="scroll"><mi>t</mi><mo>&#x0003e;</mo><mi mathvariant="italic">&#x003c4;</mi></math>, and to do so, it should be able to store and retrieve the values of the Fourier coefficients</p><div class="disp-formula" id="eqn10"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d11" display="block" overflow="scroll"><msup><mi mathvariant="bold">q</mi><mi>k</mi></msup><mo>=</mo><mfrac><mn>1</mn><mi mathvariant="italic">&#x003c4;</mi></mfrac><msubsup><mo>&#x0222b;</mo><mn>0</mn><mi mathvariant="italic">&#x003c4;</mi></msubsup><mrow><mi>d</mi><mi>t</mi><mo>&#x02009;</mo><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></msup><mo>(</mo><mi>t</mi><mo>)</mo><msup><mi>e</mi><mrow><mo>&#x02212;</mo><mi>i</mi><mn>2</mn><mi>&#x003c0;</mi><msub><mi>f</mi><mi>k</mi></msub><mi>t</mi></mrow></msup></mrow><mo>,</mo></math></div><div class="l" style="width:3.50em">[6.1]</div></div><p>or more generally, wavelet coefficients</p><div class="disp-formula" id="eqn11"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d12" display="block" overflow="scroll"><msup><mi mathvariant="bold">q</mi><mi>k</mi></msup><mo>=</mo><mfrac><mn>1</mn><mi mathvariant="italic">&#x003c4;</mi></mfrac><msubsup><mo>&#x0222b;</mo><mn>0</mn><mi mathvariant="italic">&#x003c4;</mi></msubsup><mrow><mi>d</mi><mi>t</mi><mo>&#x02009;</mo><msub><mi>W</mi><mi>k</mi></msub><mo>(</mo><mi mathvariant="italic">&#x003c4;</mi><mo>&#x02212;</mo><mi>t</mi><mo>)</mo><mo>&#x02009;</mo><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></msup><mo>(</mo><mi>t</mi><mo>)</mo><msup><mi>e</mi><mrow><mo>&#x02212;</mo><mi>i</mi><mn>2</mn><mi>&#x003c0;</mi><msub><mi>f</mi><mi>k</mi></msub><mi>t</mi></mrow></msup></mrow><mo>,</mo></math></div><div class="l" style="width:3.50em">[6.2]</div></div><p></p><p>for suitably defined window functions <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i89" display="inline" overflow="scroll"><mrow><msub><mi>W</mi><mi>i</mi></msub></mrow></math>. Then, a prediction could be made by extrapolating <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i90" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></mrow></msup><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math> using the inverse transformation</p><div class="disp-formula" id="eqn12"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d13" display="block" overflow="scroll"><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></msup><mo>(</mo><mi>t</mi><mo>+</mo><mi>&#x003b4;</mi><mo>)</mo><mo>&#x02248;</mo><mn>2</mn><munderover><mo>&#x02211;</mo><mrow><mi>k</mi><mo>=</mo><msub><mi>k</mi><mrow><mi>m</mi><mi>i</mi><mi>n</mi></mrow></msub></mrow><msub><mi>k</mi><mrow><mi>m</mi><mi>a</mi><mi>x</mi></mrow></msub></munderover><mi>Re</mi><mo>&#x0200a;</mo><mo>(</mo><msup><mi mathvariant="bold">q</mi><mi>k</mi></msup><msup><mi>e</mi><mrow><mi>i</mi><mn>2</mn><mi>&#x003c0;</mi><msub><mi>f</mi><mi>k</mi></msub><mi>&#x003b4;</mi></mrow></msup><mo>)</mo><mo>,</mo></math></div><div class="l" style="width:3.50em">[6.3]</div></div><p></p><p>for some <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i91" display="inline" overflow="scroll"><mrow><mi mathvariant="italic">&#x003b4;</mi><mo>&#x0003e;</mo><mn>0</mn></mrow></math>, which is not too large compared with <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i92" display="inline" overflow="scroll"><mi mathvariant="italic">&#x003c4;</mi></math>. However, in general, the total number of (Fourier or wavelet) coefficients <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i93" display="inline" overflow="scroll"><msup><mi mathvariant="bold">q</mi><mi>k</mi></msup></math> would be countably infinite. Therefore, any finite-size organism has to &#x0201c;decide&#x0201d; which frequencies to observe (and remember) and which ones to filter out (and &#x0201c;forget&#x0201d;).</p><p>Let us assume that the organism &#x0201c;decided&#x0201d; to only observe/remember discrete frequencies</p><div class="disp-formula" id="eqn13"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d14" display="block" overflow="scroll"><msub><mi>f</mi><mi mathvariant="italic">min</mi></msub><mo>&#x02261;</mo><msub><mi>f</mi><msub><mi>k</mi><mi mathvariant="italic">min</mi></msub></msub><mo>,</mo><mo>&#x02026;</mo><mo>,</mo><msub><mi>f</mi><msub><mi>k</mi><mi mathvariant="italic">max</mi></msub></msub><mo>&#x02261;</mo><msub><mi>f</mi><mi mathvariant="italic">max</mi></msub><mo>,</mo></math></div><div class="l" style="width:3.50em">[6.4]</div></div><p>and forget everything else. Then, to predict the state of the environment [<a href="#eqn12" rid="eqn12" class=" disp-formula"><strong>6.3</strong></a>] and as a result, minimize the loss function [<a href="#eqn7" rid="eqn7" class=" disp-formula"><strong>4.5</strong></a>], the organism should be able to store, retrieve, and adjust information about coefficients <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i94" display="inline" overflow="scroll"><msup><mi mathvariant="bold">q</mi><mi>k</mi></msup></math> in some adaptable trainable variables <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i95" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mrow><mrow><mo stretchy="true">(</mo><mi>a</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math>.</p><p>Given this simple model, we can study the flow of information between different nontrainable variables of the organism <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i96" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">x</mi><mrow><mrow><mo stretchy="true">(</mo><mi>o</mi><mo stretchy="true">)</mo></mrow></mrow></msup></mrow></math>. To this end, it is convenient to organize the variables as</p><div class="disp-formula" id="eqn14"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d15" display="block" overflow="scroll"><mi mathvariant="bold">x</mi><mo>=</mo><mrow><mo stretchy="true">(</mo><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="true">(</mo><mi>o</mi><mo stretchy="true">)</mo></mrow></msup><mo>,</mo><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></msup><mo stretchy="true">)</mo></mrow><mo>=</mo><mrow><mo stretchy="true">(</mo><msub><mi mathvariant="bold">x</mi><msub><mi>k</mi><mi mathvariant="italic">min</mi></msub></msub><mo>,</mo><mo>&#x02026;</mo><mo>,</mo><msub><mi mathvariant="bold">x</mi><msub><mi>k</mi><mi mathvariant="italic">max</mi></msub></msub><mo>,</mo><msup><mi mathvariant="bold">x</mi><mrow><mo stretchy="true">(</mo><mi>e</mi><mo stretchy="true">)</mo></mrow></msup><mo stretchy="true">)</mo></mrow><mo>,</mo></math></div><div class="l" style="width:3.50em">[6.5]</div></div><p>where</p><div class="disp-formula" id="eqn15"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d16" display="block" overflow="scroll"><msub><mi mathvariant="bold">x</mi><mi>k</mi></msub><mo>(</mo><mi>t</mi><mo>+</mo><mi>&#x003b4;</mi><mo>)</mo><mo>&#x02248;</mo><mn>2</mn><munderover><mo>&#x02211;</mo><mrow><mi>l</mi><mo>=</mo><msub><mi>k</mi><mrow><mi>m</mi><mi>i</mi><mi>n</mi></mrow></msub></mrow><mi>k</mi></munderover><mi>Re</mi><mo>&#x0200a;</mo><mo>(</mo><msup><mi mathvariant="bold">q</mi><mi>l</mi></msup><msup><mi>e</mi><mrow><mi>i</mi><mn>2</mn><mi>&#x003c0;</mi><msub><mi>f</mi><mi>l</mi></msub><mi>&#x003b4;</mi></mrow></msup><mo>)</mo><mo>,</mo></math></div><div class="l" style="width:3.50em">[6.6]</div></div><p></p><p>and assume that the relevant information about <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i97" display="inline" overflow="scroll"><mrow><msup><mi mathvariant="bold">q</mi><mi>i</mi></msup></mrow></math>&#x02019;s is stored in the adaptable trainable variables</p><div class="disp-formula" id="eqn16"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d17" display="block" overflow="scroll"><msup><mi mathvariant="bold">q</mi><mrow><mo stretchy="true">(</mo><mi>a</mi><mo stretchy="true">)</mo></mrow></msup><mo>=</mo><mrow><mo stretchy="true">(</mo><msup><mi mathvariant="bold">q</mi><msub><mi>k</mi><mi mathvariant="italic">min</mi></msub></msup><mo>,</mo><mo>&#x02026;</mo><mo>,</mo><msup><mi mathvariant="bold">q</mi><msub><mi>k</mi><mi mathvariant="italic">max</mi></msub></msup><mo stretchy="true">)</mo></mrow><mo>.</mo></math></div><div class="l" style="width:3.50em">[6.7]</div></div><p></p><p>In the estimate of <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i98" display="inline" overflow="scroll"><msub><mi mathvariant="bold">x</mi><mi>k</mi></msub><mo>(</mo><mi>t</mi><mo>+</mo><mi>&#x003b4;</mi><mo>)</mo></math> in <a href="#eqn10" rid="eqn10" class=" disp-formula">Eq. <strong>6.6</strong></a>, all the higher-frequency modes are assumed to average to zero as is often the case if we are only interested in the timescale <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i99" display="inline" overflow="scroll"><msubsup><mi>f</mi><mi>k</mi><mrow><mo>&#x02212;</mo><mn>1</mn></mrow></msubsup></math>. A better estimate can be obtained using, once again, the ideas of the renormalization group flow following the fundamental principle P4. To make learning (and thus, survival) efficient, truncation of the set of variables relevant for learning is crucial. The main point is that the higher-frequency modes can still contribute statistically, and then, an improved estimate of <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i100" display="inline" overflow="scroll"><msub><mi mathvariant="bold">x</mi><mi>k</mi></msub><mo>(</mo><mi>t</mi><mo>+</mo><mi>&#x003b4;</mi><mo>)</mo></math> would be obtained by appropriately modifying the values of the coefficients <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i101" display="inline" overflow="scroll"><msup><mi mathvariant="bold">q</mi><mi>k</mi></msup></math>. Either way, in order to make an actual prediction, the organism should first calculate <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i102" display="inline" overflow="scroll"><msub><mi mathvariant="bold">x</mi><mi>k</mi></msub><mo>(</mo><mi>t</mi><mo>+</mo><mi>&#x003b4;</mi><mo>)</mo></math> for small <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i103" display="inline" overflow="scroll"><msub><mi>f</mi><mi>k</mi></msub></math> and then, pass the result to the next level to calculate <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i104" display="inline" overflow="scroll"><msub><mi mathvariant="bold">x</mi><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub><mo>(</mo><mi>t</mi><mo>+</mo><mi>&#x003b4;</mi><mo>)</mo></math> for larger <math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="i105" display="inline" overflow="scroll"><msub><mi>f</mi><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub></math> and so on. Such computations can be described by a simple mapping</p><div class="disp-formula" id="eqn17"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" id="d18" display="block" overflow="scroll"><msub><mi mathvariant="bold">x</mi><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub><mrow><mo stretchy="true">(</mo><mi>t</mi><mo>+</mo><mi>&#x003b4;</mi><mo stretchy="true">)</mo></mrow><mo>=</mo><msub><mi mathvariant="bold">x</mi><mi>k</mi></msub><mrow><mo stretchy="true">(</mo><mi>t</mi><mo>+</mo><mi>&#x003b4;</mi><mo stretchy="true">)</mo></mrow><mo>+</mo><mn>2</mn><mi>Re</mi><mo>&#x0200a;</mo><mo>(</mo><msup><mi mathvariant="bold">q</mi><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msup><msup><mi>e</mi><mrow><mi>i</mi><mn>2</mn><mi>&#x003c0;</mi><msub><mi>f</mi><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow></msub><mi>&#x003b4;</mi></mrow></msup><mo>)</mo><mo>,</mo></math></div><div class="l" style="width:3.50em">[6.8]</div></div><p>which can be interpreted as passage of data from one layer to another in a deep, multilayer neural network (<a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2120037119/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank"><em>SI Appendix</em>, Fig. S2</a>). <a href="#eqn10" rid="eqn10" class=" disp-formula">Eq. <strong>6.8</strong></a> implies that, during the predicting phase, relevant information only flows from variables encoding low frequencies to variables encoding high frequencies but not in the reverse direction. In other words, in the process of predicting the environment, information propagates from slower variables to faster variables: that is, from genotype to phenotype or from nucleic acids to proteins (hence, the Central Dogma). Because only the fast variables change in this process, the prediction of the state of the environment is rapid, as it is indeed required to be for the organism survival. Conversely, in the process of learning the environment, information is back propagated in the opposite direction: that is, from faster to slower variables. However, this back propagation is not a microscopic reversal of the forward propagation but a distinct, much slower process (given that changes in slow variables are required) that involves mutation and selection.</p><p class="p p-last">Thus, the meaning of the generalized Central Dogma from the point of view of the learning theory&#x02014;and our theory of evolution&#x02014;is that slow dynamics (that is, evolution on a long timescale) should be mostly independent of the fast variables. In less formal terms, slow variables determine the rules of the game, and changing these rules depending on the results of some particular games would be detrimental for the organism. Optimization within the space of opportunities constrained by temporally stable rules is advantageous compared with optimization without such constraints. The trade-off between global and local optimization is a general, intrinsic property of frustrated systems (E2). For the system to function efficiently, the impact of local optimization on the global optimization should be restricted. The separation of the long-term and short-term forms of memory through different elemental bases (nucleic acids vs. proteins) serves this objective.</p></div><div id="s17" class="tsec sec"><h2 class="head no_bottom_margin" id="s17title" style="text-transform: uppercase;">7. Discussion</h2><p class="p p-first">In this work, we outline a theory of evolution on the basis of the theory of learning. The parallel between learning and biological evolution becomes obvious as soon as the mapping between the loss function and the fitness function is established (<a href="#eqn3" rid="eqn3" class=" disp-formula">Eq. <strong>4.6</strong></a>). Indeed, both processes represent movement of an evolving (learning) system on a fitness (loss function) landscape, where adaptive (learning), upward moves are most consequential, although neutral moves are most common, and downward moves also occur occasionally. However, we go beyond the obvious analogy and trace a detailed correspondence between the essential features of the evolutionary and learning processes. Arguably, the most important fundamental commonality between evolution and learning is the stratification of the trainable variables (degrees of freedom) into classes that differ by the rate of change. At least in complex environments, all learning is multilevel, and so is all selection that is relevant for the evolutionary process. The framework of evolution as learning developed here implies that evolution of biological complexity would be impossible without MLS permeating the entire history of life. Under this perspective, emergence of new levels of organization, in learning and in evolution, and in particular, MTE represent genuine phase transitions as previously suggested (<a href="#r41" rid="r41" class=" bibr popnode">41</a>). Such transitions can be analyzed consistently only in the thermodynamic limit, which is addressed in detail in the accompanying paper (<a href="#r86" rid="r86" class=" bibr popnode">86</a>).</p><p>The origin of complexity and long-term memory from simple fundamental physical laws is one of the hardest problems in all of science. One popular approach is synergetics pioneered by Haken (<a href="#r91" rid="r91" class=" bibr popnode">91</a>, <a href="#r92" rid="r92" class=" bibr popnode">92</a>) and the related nonequilibrium thermodynamics founded by Prigogine and Stengers (<a href="#r93" rid="r93" class=" bibr popnode">93</a>) that employ mathematical tools of the theory of dynamical systems, such as theory of bifurcations and analysis of attractors. However, these concepts appear to be too general and oversimplified to usefully analyze biological phenomena, which are far more complex than &#x0201c;dissipative structures&#x0201d; that are central to nonequilibrium thermodynamics, such as, for example, autowave chemical reactions.</p><p>An alternative is the approach based on the theory of spin glasses (<a href="#r43" rid="r43" class=" bibr popnode">43</a>, <a href="#r94" rid="r94" class=" bibr popnode">94</a>), which employs the mathematical apparatus of statistical physics and seems to provide a deeper insight into the origin of complexity. However, the energy landscape of spin glasses contains too many minima that are too shallow to account for long-term memory that is central to biology (<a href="#r12" rid="r12" class=" bibr popnode">12</a>, <a href="#r41" rid="r41" class=" bibr popnode">41</a>). Thus, some generalization of the spin glass concept is likely to be required for productive application in evolutionary biology (<a href="#r95" rid="r95" class=" bibr popnode">95</a>).</p><p>A popular and promising approach is self-organized criticality (SOC), a concept developed by Bak et&#x000a0;al. (<a href="#r96" rid="r96" class=" bibr popnode">96</a>, <a href="#r97" rid="r97" class=" bibr popnode">97</a>). Although relevant in biological contexts (<a href="#r12" rid="r12" class=" bibr popnode">12</a>), SOC, by definition, implies self-similarity between different levels of organization, whereas biologically relevant complexity is rather associated with distinct emergent phenomena at different spatiotemporal scales (<a href="#r90" rid="r90" class=" bibr popnode">90</a>).</p><p>A fundamental shortcoming of all these approaches is that they do not include, at least not as a major component, evolutionary concepts, such as natural selection. The framework of learning theory used here allows us to naturally unify the descriptions of physical and biological phenomena in terms of optimization by trial and error and loss (fitness) functions. Indeed, a key point of the present analysis is that most of our general principles apply to both living and nonliving systems.</p><p>The detailed correspondence between the key features of the processes of learning and biological evolution implies that this is not a simple analogy but rather, a reflection of the deep unity of evolutionary processes occurring in the universe. Indeed, separation of the relevant degrees of freedom into multiple temporal classes is ubiquitous in the universe from composite subatomic particles, such as protons, to atoms, molecules, life-forms, planetary systems, and galaxy clusters. If the entire universe is conceptualized as a neural network (<a href="#r36" rid="r36" class=" bibr popnode">36</a>), all these systems can be considered emerging from the learning dynamics. Furthermore, scale separation and renormalizability appear to be essential conditions for a universe to be observable. According to the evolution theory outlined here, any observable universe consists of systems that undergo learning or synonymously, adaptive evolution, and actually, the universe itself is such a system (<a href="#r36" rid="r36" class=" bibr popnode">36</a>). The famous dictum of Dobzhansky (<a href="#r98" rid="r98" class=" bibr popnode">98</a>), thus, can and arguably should be rephrased as &#x0201c;[n]othing in the world is comprehensible except in the light of learning.&#x0201d;</p><p>Within the theory of evolution outlined here, the difference between life and nonliving systems, however important, can be considered as one in the type and degree of optimization, so that all evolutionary phenomena can be described within the same formal framework of the theory of learning. Crucially, any complex optimization problem can be addressed only with a stochastic learning algorithm: hence, the ubiquity of selection. Origin of life can then be conceptualized within the framework of multilevel learning as we explicitly show in the accompanying paper (<a href="#r86" rid="r86" class=" bibr popnode">86</a>). The point when life begins can be naturally associated with the emergence of a distinct class of slowly changing variables that are digitized and thus, can be accurately replicated; these digital variables store and supply information for forward propagation to predict the state of the environment. In biological terms, this focal point corresponds to the advent of replicators (genomes) that carry information on the operation of reproducers within which they reside (<a href="#r99" rid="r99" class=" bibr popnode">99</a>). This is also the point when natural (Darwinian) selection takes off (<a href="#r64" rid="r64" class=" bibr popnode">64</a>). Our theory of evolution implies that this pivotal stage was preceded by evolution of &#x0201c;prelife,&#x0201d; which comprised reproducers that lacked genomes but nevertheless, were learning systems that were subject to selection for persistence. Self-reproducing micelles that harbor autocatalytic protometabolic reaction networks appear to be plausible models of such primordial reproducers (<a href="#r100" rid="r100" class=" bibr popnode">100</a>). The first replicators (RNA molecules) would evolve within these reproducers, perhaps, initially, as molecular parasites (E9) but subsequently, under selection for the ability to store, express, and share information essential for the entire system. This key step greatly increased the efficiency of evolution/learning and provided for long-term memory that persisted throughout the history of life, enabling the onset of natural selection and the unprecedented diversification of life-forms (E5). It has to be emphasized that, compared with the existing evolutionary models that explore replicator dynamics, the learning approach described here is more microscopic in that the existence of replicators is not initially assumed but rather, appears as an emergent property of multilevel learning dynamics. For learning to be efficient, the capacity of the system to add new adaptable variables is essential. In biological terms, this implies expandability of the genome (that is, the ability to add new genes), which necessitated the transition from RNA to DNA as the genome substrate given the apparent intrinsic size constraints on replicating RNA molecules. Another essential condition for efficient learning is information sharing, which in the biological context, corresponds to horizontal gene transfer. The essentiality of horizontal gene transfer at the earliest stages of life evolution is perceived as the cause of the universality of the translation machinery and genetic code in all known life-forms (<a href="#r101" rid="r101" class=" bibr popnode">101</a>). The conceptual model of the origin of life implied by our learning-based theoretical framework appears to be fully compatible with G&#x000e1;nti&#x02019;s chemoton, a model of protocell emergence and evolution based on autocatalytic reaction networks (<a href="#r102" rid="r102" class=" bibr popnode">102</a><a href="#r103" rid="r103" class=" bibr popnode"></a>&#x02013;<a href="#r104" rid="r104" class=" bibr popnode">104</a>).</p><p>The origin of life scenario within the encompassing framework of the present evolution theory, even if formulated in most general terms, implies that emergence of complexity commensurate with life is a general trend in the evolution of complex systems. At face value, this conclusion might seem to be at odds with the magnitude of complexification involved in the origin of life [suffice it to consider the complexity of the translation system (<a href="#r7" rid="r7" class=" bibr popnode">7</a>)] and the uniqueness of this event, at least on Earth and probably, on a much greater cosmic scale. Nevertheless, the origin of life appears to be an expected outcome of learning subject to the relevant constraints, such as the presence of the required chemicals in sufficient concentrations. Such constraints would make life a rare phenomenon but likely far from unique on the scale of the universe. The universe is sometimes claimed to be fine-tuned for the existence of life (<a href="#r105" rid="r105" class=" bibr popnode">105</a>). What we posit here is that the universe is self-tuned for life emergence.</p><p class="p p-last">Evidently, the analysis presented here and in the accompanying paper (<a href="#r86" rid="r86" class=" bibr popnode">86</a>) is only an outline of a theory of evolution as learning. The details and implications, including directly testable ones, remain to be worked out.</p></div><div id="sec-a.e.r" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="sec-a.e.rtitle" style="text-transform: uppercase;">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="supplementary-material-a.e.r.b"><h4>Supplementary File</h4><div class="sup-box half_rhythm" id="d64e3414"><a href="/pmc/articles/PMC8833143/bin/pnas.2120037119.sapp.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(580K, pdf)</sup></div></div></div><div id="ack-a.f.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.f.atitle" style="text-transform: uppercase;">Acknowledgments</h2><div class="sec"><p>E.V.K. is grateful to Dr. Purificacion Lopez-Garcia for essential discussions and critical reading of the manuscript. V.V. was supported in part by the Foundational Questions Institute and the Oak Ridge Institute for Science and Education. Y.I.W. and E.V.K. are supported by the Intramural Research Program of the NIH.</p></div></div><div id="fn-group-a.f.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.f.btitle" style="text-transform: uppercase;">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn3"><p class="p p-first-last">Reviewers: S.F., University of California, Irvine; and E.S., Parmenides Foundation.</p></p><p class="fn sec" id="fn4"><p class="p p-first-last">The authors declare no competing interest.</p></p><p class="fn sec" id="fn5"><p class="p p-first-last">This article contains supporting information online at <a href="https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2120037119/-/DCSupplemental" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.pnas.org/lookup/suppl/doi:10.1073/pnas.2120037119/-/DCSupplemental</a>.</p></p></div></div><div id="s18" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="s18title" style="text-transform: uppercase;">Data Availability</h2><!--/article/back/sec/--><p class="p p-first-last">There are no data underlying this work.</p></div><div id="ref-list-a.f.d" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.f.dtitle" style="text-transform: uppercase;">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="r1">1. <span class="mixed-citation">Morange M., <span class="ref-title">The recent evolution of the question &#x0201c;What is life&#x0201d;?</span>
<span class="ref-journal">Hist. Philos. Life Sci.</span>
<span class="ref-vol">34</span>, 425&#x02013;436 (2012). [<a href="https://pubmed.ncbi.nlm.nih.gov/23316570" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hist.+Philos.+Life+Sci.&amp;title=The+recent+evolution+of+the+question+&#x0201c;What+is+life&#x0201d;?&amp;volume=34&amp;publication_year=2012&amp;pages=425-436&amp;pmid=23316570&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r2">2. <span class="mixed-citation">Benner S. A., <span class="ref-title">Defining life</span>. <span class="ref-journal">Astrobiology</span>
<span class="ref-vol">10</span>, 1021&#x02013;1030 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3005285/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21162682" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Astrobiology&amp;title=Defining+life&amp;volume=10&amp;publication_year=2010&amp;pages=1021-1030&amp;pmid=21162682&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r3">3. <span class="mixed-citation">Trifonov E. N., <span class="ref-title">Vocabulary of definitions of life suggests a definition</span>. <span class="ref-journal">J. Biomol. Struct. Dyn.</span>
<span class="ref-vol">29</span>, 259&#x02013;266 (2011). [<a href="https://pubmed.ncbi.nlm.nih.gov/21875147" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Biomol.+Struct.+Dyn.&amp;title=Vocabulary+of+definitions+of+life+suggests+a+definition&amp;volume=29&amp;publication_year=2011&amp;pages=259-266&amp;pmid=21875147&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r4">4. <span class="mixed-citation">Koonin E. V., <span class="ref-journal">The Logic of Chance: The Nature and Origin of Biological Evolution</span> (FT Press, Upper Saddle River, NJ, 2011). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Logic+of+Chance:+The+Nature+and+Origin+of+Biological+Evolution&amp;publication_year=2011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r5">5. <span class="mixed-citation">Eigen M., <span class="ref-title">Selforganization of matter and the evolution of biological macromolecules</span>. <span class="ref-journal">Naturwissenschaften</span>
<span class="ref-vol">58</span>, 465&#x02013;523 (1971). [<a href="https://pubmed.ncbi.nlm.nih.gov/4942363" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Naturwissenschaften&amp;title=Selforganization+of+matter+and+the+evolution+of+biological+macromolecules&amp;volume=58&amp;publication_year=1971&amp;pages=465-523&amp;pmid=4942363&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r6">6. <span class="mixed-citation">Penny D., <span class="ref-title">An interpretative review of the origin of life research</span>. <span class="ref-journal">Biol. Philos.</span>
<span class="ref-vol">20</span>, 633&#x02013;671 (2005). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Philos.&amp;title=An+interpretative+review+of+the+origin+of+life+research&amp;volume=20&amp;publication_year=2005&amp;pages=633-671&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r7">7. <span class="mixed-citation">Wolf Y. I., Koonin E. V., <span class="ref-title">On the origin of the translation system and the genetic code in the RNA world by means of natural selection, exaptation, and subfunctionalization</span>. <span class="ref-journal">Biol. Direct</span>
<span class="ref-vol">2</span>, 14 (2007). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1894784/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17540026" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Direct&amp;title=On+the+origin+of+the+translation+system+and+the+genetic+code+in+the+RNA+world+by+means+of+natural+selection,+exaptation,+and+subfunctionalization&amp;volume=2&amp;publication_year=2007&amp;pages=14&amp;pmid=17540026&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r8">8. <span class="mixed-citation">Hartl D. L., Clark A. G., <span class="ref-journal">Principles of Population Genetics</span> (Sinauer Associates, Sunderland, MA, ed. 4, 2006). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Principles+of+Population+Genetics&amp;publication_year=2006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r9">9. <span class="mixed-citation">Lynch M., <span class="ref-journal">The Origins of Genome Architecture</span> (Sinauer Associates, Sunderland, MA, 2007). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Origins+of+Genome+Architecture&amp;publication_year=2007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r10">10. <span class="mixed-citation">Bernardes J. P., et al., <span class="ref-title">The evolution of convex trade-offs enables the transition towards multicellularity</span>. <span class="ref-journal">Nat. Commun.</span>
<span class="ref-vol">12</span>, 4222 (2021). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8270964/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34244514" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=The+evolution+of+convex+trade-offs+enables+the+transition+towards+multicellularity&amp;volume=12&amp;publication_year=2021&amp;pages=4222&amp;pmid=34244514&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r11">11. <span class="mixed-citation">Michod R. E., <span class="ref-title">The group covariance effect and fitness trade-offs during evolutionary transitions in individuality</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">103</span>, 9113&#x02013;9117 (2006). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1482575/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16751277" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=The+group+covariance+effect+and+fitness+trade-offs+during+evolutionary+transitions+in+individuality&amp;volume=103&amp;publication_year=2006&amp;pages=9113-9117&amp;pmid=16751277&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r12">12. <span class="mixed-citation">Wolf Y. I., Katsnelson M. I., Koonin E. V., <span class="ref-title">Physical foundations of biological complexity</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">115</span>, E8678&#x02013;E8687 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6140470/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30150417" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Physical+foundations+of+biological+complexity&amp;volume=115&amp;publication_year=2018&amp;pages=E8678-E8687&amp;pmid=30150417&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r13">13. <span class="mixed-citation">Forterre P., Prangishvili D., <span class="ref-title">The great billion-year war between ribosome- and capsid-encoding organisms (cells and viruses) as the major source of evolutionary novelties</span>. <span class="ref-journal">Ann. N. Y. Acad. Sci.</span>
<span class="ref-vol">1178</span>, 65&#x02013;77 (2009). [<a href="https://pubmed.ncbi.nlm.nih.gov/19845628" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ann.+N.+Y.+Acad.+Sci.&amp;title=The+great+billion-year+war+between+ribosome-+and+capsid-encoding+organisms+(cells+and+viruses)+as+the+major+source+of+evolutionary+novelties&amp;volume=1178&amp;publication_year=2009&amp;pages=65-77&amp;pmid=19845628&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r14">14. <span class="mixed-citation">Koonin E. V., <span class="ref-title">Viruses and mobile elements as drivers of evolutionary transitions</span>. <span class="ref-journal">Philos. Trans. R. Soc. Lond. B Biol. Sci.</span>
<span class="ref-vol">371</span>, 20150442 (2016). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4958936/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27431520" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Philos.+Trans.+R.+Soc.+Lond.+B+Biol.+Sci.&amp;title=Viruses+and+mobile+elements+as+drivers+of+evolutionary+transitions&amp;volume=371&amp;publication_year=2016&amp;pages=20150442&amp;pmid=27431520&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r15">15. <span class="mixed-citation">Koonin E. V., Makarova K. S., Wolf Y. I., Krupovic M., <span class="ref-title">Evolutionary entanglement of mobile genetic elements and host defence systems: Guns for hire</span>. <span class="ref-journal">Nat. Rev. Genet.</span>
<span class="ref-vol">21</span>, 119&#x02013;131 (2020). [<a href="https://pubmed.ncbi.nlm.nih.gov/31611667" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Evolutionary+entanglement+of+mobile+genetic+elements+and+host+defence+systems:+Guns+for+hire&amp;volume=21&amp;publication_year=2020&amp;pages=119-131&amp;pmid=31611667&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r16">16. <span class="mixed-citation">Koonin E. V., Wolf Y. I., Katsnelson M. I., <span class="ref-title">Inevitability of the emergence and persistence of genetic parasites caused by thermodynamic instability of parasite-free states</span>. <span class="ref-journal">Biol. Direct</span>
<span class="ref-vol">12</span>, 31 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5715634/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29202832" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biol.+Direct&amp;title=Inevitability+of+the+emergence+and+persistence+of+genetic+parasites+caused+by+thermodynamic+instability+of+parasite-free+states&amp;volume=12&amp;publication_year=2017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r17">17. <span class="mixed-citation">Nelson P., Masel J., <span class="ref-title">Intercellular competition and the inevitability of multicellular aging</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">114</span>, 12982&#x02013;12987 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5724245/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29087299" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Intercellular+competition+and+the+inevitability+of+multicellular+aging&amp;volume=114&amp;publication_year=2017&amp;pages=12982-12987&amp;pmid=29087299&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r18">18. <span class="mixed-citation">Boddy A. M., Huang W., Aktipis A., <span class="ref-title">Life history trade-offs in tumors</span>. <span class="ref-journal">Curr. Pathobiol. Rep.</span>
<span class="ref-vol">6</span>, 201&#x02013;207 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6290708/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30595969" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Pathobiol.+Rep.&amp;title=Life+history+trade-offs+in+tumors&amp;volume=6&amp;publication_year=2018&amp;pages=201-207&amp;pmid=30595969&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r19">19. <span class="mixed-citation">Szathm&#x000e1;ry E., Smith J. M., <span class="ref-title">The major evolutionary transitions</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">374</span>, 227&#x02013;232 (1995). [<a href="https://pubmed.ncbi.nlm.nih.gov/7885442" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+major+evolutionary+transitions&amp;volume=374&amp;publication_year=1995&amp;pages=227-232&amp;pmid=7885442&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r20">20. <span class="mixed-citation">Maynard Smith J., Szathmary E., <span class="ref-journal">The Major Transitions in Evolution</span> (Oxford University Press, Oxford, United Kingdom, 1997). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Major+Transitions+in+Evolution&amp;publication_year=1997&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r21">21. <span class="mixed-citation">Szathm&#x000e1;ry E., <span class="ref-title">Toward major evolutionary transitions theory 2.0</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">112</span>, 10104&#x02013;10111 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4547294/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25838283" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Toward+major+evolutionary+transitions+theory+2.0&amp;volume=112&amp;publication_year=2015&amp;pages=10104-10111&amp;pmid=25838283&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r22">22. <span class="mixed-citation">Cz&#x000e9;gel D., Zachar I., Szathm&#x000e1;ry E., <span class="ref-title">Multilevel selection as Bayesian inference, major transitions in individuality as structure learning</span>. <span class="ref-journal">R. Soc. Open Sci.</span>
<span class="ref-vol">6</span>, 190202 (2019). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6731722/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31598234" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=R.+Soc.+Open+Sci.&amp;title=Multilevel+selection+as+Bayesian+inference,+major+transitions+in+individuality+as+structure+learning&amp;volume=6&amp;publication_year=2019&amp;pages=190202&amp;pmid=31598234&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r23">23. <span class="mixed-citation">Okashi S., <span class="ref-journal">Evolution and the Levels of Selection</span> (Oxford University Press, Oxford, United Kingdom, 2006). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Evolution+and+the+Levels+of+Selection&amp;publication_year=2006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r24">24. <span class="mixed-citation">Brunet T. D., Doolittle W. F., <span class="ref-title">Multilevel selection theory and the evolutionary functions of transposable elements</span>. <span class="ref-journal">Genome Biol. Evol.</span>
<span class="ref-vol">7</span>, 2445&#x02013;2457 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4558868/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26253318" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.+Evol.&amp;title=Multilevel+selection+theory+and+the+evolutionary+functions+of+transposable+elements&amp;volume=7&amp;publication_year=2015&amp;pages=2445-2457&amp;pmid=26253318&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r25">25. <span class="mixed-citation">Gardner A., <span class="ref-title">The genetical theory of multilevel selection</span>. <span class="ref-journal">J. Evol. Biol.</span>
<span class="ref-vol">28</span>, 305&#x02013;319 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4415573/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25475922" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Evol.+Biol.&amp;title=The+genetical+theory+of+multilevel+selection&amp;volume=28&amp;publication_year=2015&amp;pages=305-319&amp;pmid=25475922&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r26">26. <span class="mixed-citation">Jeler C., <span class="ref-title">Explanatory goals and explanatory means in multilevel selection theory</span>. <span class="ref-journal">Hist. Philos. Life Sci.</span>
<span class="ref-vol">42</span>, 36 (2020). [<a href="https://pubmed.ncbi.nlm.nih.gov/32779040" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hist.+Philos.+Life+Sci.&amp;title=Explanatory+goals+and+explanatory+means+in+multilevel+selection+theory&amp;volume=42&amp;publication_year=2020&amp;pages=36&amp;pmid=32779040&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r27">27. <span class="mixed-citation">Wade M. J., et al., <span class="ref-title">Multilevel and kin selection in a connected world</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">463</span>, E8&#x02013;E9 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3151728/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20164866" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Multilevel+and+kin+selection+in+a+connected+world&amp;volume=463&amp;publication_year=2010&amp;pages=E8-E9&amp;pmid=20164866&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r28">28. <span class="mixed-citation">M&#x000e1;rquez-Zacar&#x000ed;as P., et al., <span class="ref-title">Evolution of cellular differentiation: From hypotheses to models</span>. <span class="ref-journal">Trends Ecol. Evol.</span>
<span class="ref-vol">36</span>, 49&#x02013;60 (2021). [<a href="https://pubmed.ncbi.nlm.nih.gov/32829916" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Ecol.+Evol.&amp;title=Evolution+of+cellular+differentiation:+From+hypotheses+to+models&amp;volume=36&amp;publication_year=2021&amp;pages=49-60&amp;pmid=32829916&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r29">29. <span class="mixed-citation">Niklas K. J., Newman S. A., <span class="ref-title">The many roads to and from multicellularity</span>. <span class="ref-journal">J. Exp. Bot.</span>
<span class="ref-vol">71</span>, 3247&#x02013;3253 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7289717/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31819969" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Bot.&amp;title=The+many+roads+to+and+from+multicellularity&amp;volume=71&amp;publication_year=2020&amp;pages=3247-3253&amp;pmid=31819969&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r30">30. <span class="mixed-citation">Kramer J., Meunier J., <span class="ref-title">Kin and multilevel selection in social evolution: A never-ending controversy?</span>
<span class="ref-journal">F1000 Res.</span>
<span class="ref-vol">5</span>, 5 (2016). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4850877/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27158472" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=F1000+Res.&amp;title=Kin+and+multilevel+selection+in+social+evolution:+A+never-ending+controversy?&amp;volume=5&amp;publication_year=2016&amp;pages=5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r31">31. <span class="mixed-citation">Edwards S. F., Anderson P. W., <span class="ref-title">Theory of spin glasses</span>. <span class="ref-journal">J. Phys. F Met. Phys.</span>
<span class="ref-vol">5</span>, 965&#x02013;974 (1975). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Phys.+F+Met.+Phys.&amp;title=Theory+of+spin+glasses&amp;volume=5&amp;publication_year=1975&amp;pages=965-974&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r32">32. <span class="mixed-citation">Fischer K. H., Hertz J. A., <span class="ref-journal">Spin Glasses</span> (Cambridge University Press, Cambridge, United Kingdom, 1993). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Spin+Glasses&amp;publication_year=1993&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r33">33. <span class="mixed-citation">McLeish T. C., <span class="ref-title">Are there ergodic limits to evolution? Ergodic exploration of genome space and convergence</span>. <span class="ref-journal">Interface Focus</span>
<span class="ref-vol">5</span>, 20150041 (2015). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4633858/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26640648" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Interface+Focus&amp;title=Are+there+ergodic+limits+to+evolution?+Ergodic+exploration+of+genome+space+and+convergence&amp;volume=5&amp;publication_year=2015&amp;pages=20150041&amp;pmid=26640648&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r34">34. <span class="mixed-citation">Schroedinger E., <span class="ref-journal">What is Life? The Physical Aspect of the Living Cell</span> (Trinity College Press, Dublin, Ireland, 1944). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=What+is+Life?+The+Physical+Aspect+of+the+Living+Cell&amp;publication_year=1944&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r35">35. <span class="mixed-citation">Vanchurin V., <span class="ref-title">Towards a theory of machine learning</span>. <span class="ref-journal">Mach. Learn. Sci. Technol.</span>
<span class="ref-vol">2</span>, 035012 (2021). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mach.+Learn.+Sci.+Technol.&amp;title=Towards+a+theory+of+machine+learning&amp;volume=2&amp;publication_year=2021&amp;pages=035012&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r36">36. <span class="mixed-citation">Vanchurin V., <span class="ref-title">The world as a neural network</span>. <span class="ref-journal">Entropy (Basel)</span>
<span class="ref-vol">22</span>, E1210 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7712105/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33286978" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Entropy+(Basel)&amp;title=The+world+as+a+neural+network&amp;volume=22&amp;publication_year=2020&amp;pages=E1210&amp;pmid=33286978&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r37">37. <span class="mixed-citation">Valiant L., <span class="ref-journal">Probably Approximately Correct: Nature&#x02019;s Algorithms for Learning and Prospering in a Complex World</span> (Basic Books, New York, NY, ed. 1, 2013). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Probably+Approximately+Correct:+Nature&#x02019;s+Algorithms+for+Learning+and+Prospering+in+a+Complex+World&amp;publication_year=2013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r38">38. <span class="mixed-citation">Watson R. A., Szathm&#x000e1;ry E., <span class="ref-title">How can evolution learn?</span>
<span class="ref-journal">Trends Ecol. Evol.</span>
<span class="ref-vol">31</span>, 147&#x02013;157 (2016). [<a href="https://pubmed.ncbi.nlm.nih.gov/26705684" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Ecol.+Evol.&amp;title=How+can+evolution+learn?&amp;volume=31&amp;publication_year=2016&amp;pages=147-157&amp;pmid=26705684&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r39">39. <span class="mixed-citation">Benfatto G., Gallavotti G., <span class="ref-journal">Renormalization Group (Physics Notes Book 1)</span> (Princeton University Press, Princeton, NJ, 2020). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Renormalization+Group+(Physics+Notes+Book+1)&amp;publication_year=2020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r40">40. <span class="mixed-citation">Goldenfeld N., <span class="ref-journal">Lectures on Phase Transitions and the Renormalization Group (Frontiers in Physics)</span> (Addison-Wesley, New York, NY, 1972). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Lectures+on+Phase+Transitions+and+the+Renormalization+Group+(Frontiers+in+Physics)&amp;publication_year=1972&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r41">41. <span class="mixed-citation">Katsnelson M. I., Wolf Y. I., Koonin E. V., <span class="ref-title">Towards physical principles of biological evolution</span>. <span class="ref-journal">Phys. Scr.</span>
<span class="ref-vol">93</span>, 043001 (2018). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phys.+Scr.&amp;title=Towards+physical+principles+of+biological+evolution&amp;volume=93&amp;publication_year=2018&amp;pages=043001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r42">42. <span class="mixed-citation">Laughlin R. B., Pines D., <span class="ref-title">The theory of everything</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">97</span>, 28&#x02013;31 (2000). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC26610/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10618365" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=The+theory+of+everything&amp;volume=97&amp;publication_year=2000&amp;pages=28-31&amp;pmid=10618365&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r43">43. <span class="mixed-citation">Laughlin R. B., Pines D., Schmalian J., Stojkovic B. P., Wolynes P., <span class="ref-title">The middle way</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">97</span>, 32&#x02013;37 (2000). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC26611/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10618366" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=The+middle+way&amp;volume=97&amp;publication_year=2000&amp;pages=32-37&amp;pmid=10618366&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r44">44. <span class="mixed-citation">Darwin C., <span class="ref-journal">On the Origin of Species</span> (A. F. Murray, London, United Kingdom, 1859). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=On+the+Origin+of+Species&amp;publication_year=1859&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r45">45. <span class="mixed-citation">Dobzhansky T., <span class="ref-journal">Genetics and the Origin of Species</span> (Columbia University Press, New York, NY, ed. 2, 1951). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Genetics+and+the+Origin+of+Species&amp;publication_year=1951&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r46">46. <span class="mixed-citation">Doolittle W. F., Inkpen S. A., <span class="ref-title">Processes and patterns of interaction as units of selection: An introduction to ITSNTS thinking</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">115</span>, 4006&#x02013;4014 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5910863/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29581311" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Processes+and+patterns+of+interaction+as+units+of+selection:+An+introduction+to+ITSNTS+thinking&amp;volume=115&amp;publication_year=2018&amp;pages=4006-4014&amp;pmid=29581311&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r47">47. <span class="mixed-citation">Lem S., <span class="ref-journal">Solaris</span> (Pro Auctore Wojciech Zemek, ed. 2, 2014). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Solaris&amp;publication_year=2014&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r48">48. <span class="mixed-citation">Shelton D. E., Michod R. E., <span class="ref-title">Group and individual selection during evolutionary transitions in individuality: Meanings and partitions</span>. <span class="ref-journal">Philos. Trans. R. Soc. Lond. B Biol. Sci.</span>
<span class="ref-vol">375</span>, 20190364 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7133510/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32146883" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Philos.+Trans.+R.+Soc.+Lond.+B+Biol.+Sci.&amp;title=Group+and+individual+selection+during+evolutionary+transitions+in+individuality:+Meanings+and+partitions&amp;volume=375&amp;publication_year=2020&amp;pages=20190364&amp;pmid=32146883&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r49">49. <span class="mixed-citation">Bakhtin Y., Katsnelson M. I., Wolf Y. I., Koonin E. V., <span class="ref-title">Evolution in the weak-mutation limit: Stasis periods punctuated by fast transitions between saddle points on the fitness landscape</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">118</span>, e2015665118 (2021). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7848522/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33472973" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Evolution+in+the+weak-mutation+limit:+Stasis+periods+punctuated+by+fast+transitions+between+saddle+points+on+the+fitness+landscape&amp;volume=118&amp;publication_year=2021&amp;pages=e2015665118&amp;pmid=33472973&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r50">50. <span class="mixed-citation">Koonin E. V., Wolf Y. I., <span class="ref-title">Constraints and plasticity in genome and molecular-phenome evolution</span>. <span class="ref-journal">Nat. Rev. Genet.</span>
<span class="ref-vol">11</span>, 487&#x02013;498 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3273317/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20548290" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Constraints+and+plasticity+in+genome+and+molecular-phenome+evolution&amp;volume=11&amp;publication_year=2010&amp;pages=487-498&amp;pmid=20548290&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r51">51. <span class="mixed-citation">Wright S., <span class="ref-journal">Adaptation and Selection. Genetics, Paleontology and Evolution</span> (Princeton University Press, Princeton, NJ, 1949). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Adaptation+and+Selection.+Genetics,+Paleontology+and+Evolution&amp;publication_year=1949&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r52">52. <span class="mixed-citation">Lynch M., et al., <span class="ref-title">Genetic drift, selection and the evolution of the mutation rate</span>. <span class="ref-journal">Nat. Rev. Genet.</span>
<span class="ref-vol">17</span>, 704&#x02013;714 (2016). [<a href="https://pubmed.ncbi.nlm.nih.gov/27739533" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Genetic+drift,+selection+and+the+evolution+of+the+mutation+rate&amp;volume=17&amp;publication_year=2016&amp;pages=704-714&amp;pmid=27739533&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r53">53. <span class="mixed-citation">Gavrilets S., <span class="ref-journal">Fitness Landscapes and the Origin of Species</span> (Princeton University Press, Princeton, NJ, 2004). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Fitness+Landscapes+and+the+Origin+of+Species&amp;publication_year=2004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r54">54. <span class="mixed-citation">Svensson E., Calsbeek R., <span class="ref-journal">The Adaptive Landscape in Evolutionary Biology</span> (Oxford University Press, Oxford, United Kingdom, 2012). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Adaptive+Landscape+in+Evolutionary+Biology&amp;publication_year=2012&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r55">55. <span class="mixed-citation">Joyce G. F., <span class="ref-title">The antiquity of RNA-based evolution</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">418</span>, 214&#x02013;221 (2002). [<a href="https://pubmed.ncbi.nlm.nih.gov/12110897" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+antiquity+of+RNA-based+evolution&amp;volume=418&amp;publication_year=2002&amp;pages=214-221&amp;pmid=12110897&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r56">56. <span class="mixed-citation">Joyce G. F., Szostak J. W., <span class="ref-title">Protocells and RNA self-replication</span>. <span class="ref-journal">Cold Spring Harb. Perspect. Biol.</span>
<span class="ref-vol">10</span>, a034801 (2018). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6120706/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30181195" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb.+Perspect.+Biol.&amp;title=Protocells+and+RNA+self-replication&amp;volume=10&amp;publication_year=2018&amp;pages=a034801&amp;pmid=30181195&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r57">57. <span class="mixed-citation">Szathm&#x000e1;ry E., <span class="ref-title">The evolution of replicators</span>. <span class="ref-journal">Philos. Trans. R. Soc. Lond. B Biol. Sci.</span>
<span class="ref-vol">355</span>, 1669&#x02013;1676 (2000). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1692888/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11127914" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Philos.+Trans.+R.+Soc.+Lond.+B+Biol.+Sci.&amp;title=The+evolution+of+replicators&amp;volume=355&amp;publication_year=2000&amp;pages=1669-1676&amp;pmid=11127914&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r58">58. <span class="mixed-citation">Szathm&#x000e1;ry E., Maynard Smith J., <span class="ref-title">From replicators to reproducers: The first major transitions leading to life</span>. <span class="ref-journal">J. Theor. Biol.</span>
<span class="ref-vol">187</span>, 555&#x02013;571 (1997). [<a href="https://pubmed.ncbi.nlm.nih.gov/9299299" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Theor.+Biol.&amp;title=From+replicators+to+reproducers:+The+first+major+transitions+leading+to+life&amp;volume=187&amp;publication_year=1997&amp;pages=555-571&amp;pmid=9299299&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r59">59. <span class="mixed-citation">Adamski P., et al., <span class="ref-title">From self-replication to replicator systems en route to de novo life</span>. <span class="ref-journal">Nat. Rev. Chem.</span>
<span class="ref-vol">4</span>, 386&#x02013;403 (2020). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Chem.&amp;title=From+self-replication+to+replicator+systems+en+route+to+de+novo+life&amp;volume=4&amp;publication_year=2020&amp;pages=386-403&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r60">60. <span class="mixed-citation">Crick F., <span class="ref-title">Central dogma of molecular biology</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">227</span>, 561&#x02013;563 (1970). [<a href="https://pubmed.ncbi.nlm.nih.gov/4913914" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Central+dogma+of+molecular+biology&amp;volume=227&amp;publication_year=1970&amp;pages=561-563&amp;pmid=4913914&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r61">61. <span class="mixed-citation">Davis M., <span class="ref-journal">The Universal Computer: The Road from Leibniz to Turing</span> (W. W. Norton &#x00026; Co., New York, NY, 2010). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Universal+Computer:+The+Road+from+Leibniz+to+Turing&amp;publication_year=2010&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r62">62. <span class="mixed-citation">Von Neumann J., First draft of a report on the EDVAC (1945). <a href="https://web.archive.org/web/20130314123032/qss.stanford.edu/~godfrey/vonNeumann/vnedvac.pdf" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://web.archive.org/web/20130314123032/qss.stanford.edu/~godfrey/vonNeumann/vnedvac.pdf</a>. Accessed 27 September 2021.</span></div><div class="ref-cit-blk half_rhythm" id="r63">63. <span class="mixed-citation">Kun A., Santos M., Szathm&#x000e1;ry E., <span class="ref-title">Real ribozymes suggest a relaxed error threshold</span>. <span class="ref-journal">Nat. Genet.</span>
<span class="ref-vol">37</span>, 1008&#x02013;1011 (2005). [<a href="https://pubmed.ncbi.nlm.nih.gov/16127452" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Real+ribozymes+suggest+a+relaxed+error+threshold&amp;volume=37&amp;publication_year=2005&amp;pages=1008-1011&amp;pmid=16127452&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r64">64. <span class="mixed-citation">Woese C., <span class="ref-title">The universal ancestor</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">95</span>, 6854&#x02013;6859 (1998). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC22660/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9618502" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=The+universal+ancestor&amp;volume=95&amp;publication_year=1998&amp;pages=6854-6859&amp;pmid=9618502&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r65">65. <span class="mixed-citation">Woese C. R., <span class="ref-title">On the evolution of cells</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">99</span>, 8742&#x02013;8747 (2002). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC124369/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12077305" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=On+the+evolution+of+cells&amp;volume=99&amp;publication_year=2002&amp;pages=8742-8747&amp;pmid=12077305&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r66">66. <span class="mixed-citation">Smith J. M., <span class="ref-title">Natural selection and the concept of a protein space</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">225</span>, 563&#x02013;564 (1970). [<a href="https://pubmed.ncbi.nlm.nih.gov/5411867" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Natural+selection+and+the+concept+of+a+protein+space&amp;volume=225&amp;publication_year=1970&amp;pages=563-564&amp;pmid=5411867&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r67">67. <span class="mixed-citation">Iranzo J., Cuesta J. A., Manrubia S., Katsnelson M. I., Koonin E. V., <span class="ref-title">Disentangling the effects of selection and loss bias on gene dynamics</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">114</span>, E5616&#x02013;E5624 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5514749/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28652353" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Disentangling+the+effects+of+selection+and+loss+bias+on+gene+dynamics&amp;volume=114&amp;publication_year=2017&amp;pages=E5616-E5624&amp;pmid=28652353&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r68">68. <span class="mixed-citation">Sagan L., <span class="ref-title">On the origin of mitosing cells</span>. <span class="ref-journal">J. Theor. Biol.</span>
<span class="ref-vol">14</span>, 255&#x02013;274 (1967). [<a href="https://pubmed.ncbi.nlm.nih.gov/11541392" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Theor.+Biol.&amp;title=On+the+origin+of+mitosing+cells&amp;volume=14&amp;publication_year=1967&amp;pages=255-274&amp;pmid=11541392&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r69">69. <span class="mixed-citation">Embley T. M., Martin W., <span class="ref-title">Eukaryotic evolution, changes and challenges</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">440</span>, 623&#x02013;630 (2006). [<a href="https://pubmed.ncbi.nlm.nih.gov/16572163" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Eukaryotic+evolution,+changes+and+challenges&amp;volume=440&amp;publication_year=2006&amp;pages=623-630&amp;pmid=16572163&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r70">70. <span class="mixed-citation">Iranzo J., Lobkovsky A. E., Wolf Y. I., Koonin E. V., <span class="ref-title">Virus-host arms race at the joint origin of multicellularity and programmed cell death</span>. <span class="ref-journal">Cell Cycle</span>
<span class="ref-vol">13</span>, 3083&#x02013;3088 (2014). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4615056/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25486567" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Cycle&amp;title=Virus-host+arms+race+at+the+joint+origin+of+multicellularity+and+programmed+cell+death&amp;volume=13&amp;publication_year=2014&amp;pages=3083-3088&amp;pmid=25486567&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r71">71. <span class="mixed-citation">Nedelcu A. M., Driscoll W. W., Durand P. M., Herron M. D., Rashidi A., <span class="ref-title">On the paradigm of altruistic suicide in the unicellular world</span>. <span class="ref-journal">Evolution</span>
<span class="ref-vol">65</span>, 3&#x02013;20 (2011). [<a href="https://pubmed.ncbi.nlm.nih.gov/20722725" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=On+the+paradigm+of+altruistic+suicide+in+the+unicellular+world&amp;volume=65&amp;publication_year=2011&amp;pages=3-20&amp;pmid=20722725&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r72">72. <span class="mixed-citation">Durand P. M., <span class="ref-journal">The Evolutionary Origins of Life and Death</span> (University of Chicago Press, Chicago, IL, 2021). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Evolutionary+Origins+of+Life+and+Death&amp;publication_year=2021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r73">73. <span class="mixed-citation">Koonin E. V., Aravind L., <span class="ref-title">Origin and evolution of eukaryotic apoptosis: The bacterial connection</span>. <span class="ref-journal">Cell Death Differ.</span>
<span class="ref-vol">9</span>, 394&#x02013;404 (2002). [<a href="https://pubmed.ncbi.nlm.nih.gov/11965492" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death+Differ.&amp;title=Origin+and+evolution+of+eukaryotic+apoptosis:+The+bacterial+connection&amp;volume=9&amp;publication_year=2002&amp;pages=394-404&amp;pmid=11965492&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r74">74. <span class="mixed-citation">Haykin S. S., <span class="ref-journal">Neural Networks: A Comprehensive Foundation</span> (Prentice Hall, New York, NY, 1999). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Neural+Networks:+A+Comprehensive+Foundation&amp;publication_year=1999&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r75">75. <span class="mixed-citation">Galushkin A. I., <span class="ref-journal">Neural Networks Theory</span> (Springer, New York, NY, 2007). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Neural+Networks+Theory&amp;publication_year=2007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r76">76. <span class="mixed-citation">Kimura M., <span class="ref-journal">The Neutral Theory of Molecular Evolution</span> (Cambridge University Press, Cambridge, United Kingdom, 1983). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Neutral+Theory+of+Molecular+Evolution&amp;publication_year=1983&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r77">77. <span class="mixed-citation">Nei M., <span class="ref-title">Selectionism and neutralism in molecular evolution</span>. <span class="ref-journal">Mol. Biol. Evol.</span>
<span class="ref-vol">22</span>, 2318&#x02013;2342 (2005). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1513187/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16120807" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Biol.+Evol.&amp;title=Selectionism+and+neutralism+in+molecular+evolution&amp;volume=22&amp;publication_year=2005&amp;pages=2318-2342&amp;pmid=16120807&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r78">78. <span class="mixed-citation">Reddy G., Desai M. M., <span class="ref-title">Global epistasis emerges from a generic model of a complex trait</span>. <span class="ref-journal">eLife</span>
<span class="ref-vol">10</span>, e64740 (2021). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8057814/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33779543" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Global+epistasis+emerges+from+a+generic+model+of+a+complex+trait&amp;volume=10&amp;publication_year=2021&amp;pages=e64740&amp;pmid=33779543&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r79">79. <span class="mixed-citation">Sailer Z. R., Harms M. J., <span class="ref-title">Molecular ensembles make evolution unpredictable</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">114</span>, 11938&#x02013;11943 (2017). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5691298/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29078365" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Molecular+ensembles+make+evolution+unpredictable&amp;volume=114&amp;publication_year=2017&amp;pages=11938-11943&amp;pmid=29078365&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r80">80. <span class="mixed-citation">Wagner A., <span class="ref-title">Neutralism and selectionism: A network-based reconciliation</span>. <span class="ref-journal">Nat. Rev. Genet.</span>
<span class="ref-vol">9</span>, 965&#x02013;974 (2008). [<a href="https://pubmed.ncbi.nlm.nih.gov/18957969" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Neutralism+and+selectionism:+A+network-based+reconciliation&amp;volume=9&amp;publication_year=2008&amp;pages=965-974&amp;pmid=18957969&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r81">81. <span class="mixed-citation">Miller J. H., <span class="ref-journal">The Operon</span> (Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1980). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Operon&amp;publication_year=1980&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r82">82. <span class="mixed-citation">Mej&#x000ed;a-Almonte C., et al., <span class="ref-title">Redefining fundamental concepts of transcription initiation in bacteria</span>. <span class="ref-journal">Nat. Rev. Genet.</span>
<span class="ref-vol">21</span>, 699&#x02013;714 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7990032/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32665585" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Genet.&amp;title=Redefining+fundamental+concepts+of+transcription+initiation+in+bacteria&amp;volume=21&amp;publication_year=2020&amp;pages=699-714&amp;pmid=32665585&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r83">83. <span class="mixed-citation">Lynch M., <span class="ref-title">Evolution of the mutation rate</span>. <span class="ref-journal">Trends Genet.</span>
<span class="ref-vol">26</span>, 345&#x02013;352 (2010). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2910838/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20594608" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Genet.&amp;title=Evolution+of+the+mutation+rate&amp;volume=26&amp;publication_year=2010&amp;pages=345-352&amp;pmid=20594608&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r84">84. <span class="mixed-citation">Friston K., Kilner J., Harrison L., <span class="ref-title">A free energy principle for the brain</span>. <span class="ref-journal">J. Physiol. Paris</span>
<span class="ref-vol">100</span>, 70&#x02013;87 (2006). [<a href="https://pubmed.ncbi.nlm.nih.gov/17097864" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Physiol.+Paris&amp;title=A+free+energy+principle+for+the+brain&amp;volume=100&amp;publication_year=2006&amp;pages=70-87&amp;pmid=17097864&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r85">85. <span class="mixed-citation">Crow J. F., Kimura M., <span class="ref-journal">Introduction to Population Genetics Theory</span> (Harper and Row, New York, NY, 1970). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Introduction+to+Population+Genetics+Theory&amp;publication_year=1970&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r86">86. <span class="mixed-citation">Vanchurin V., Wolf Y. I., Koonin E. V., Katsnelson M. I., <span class="ref-title">Thermodynamics of evolution and the origin of life</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">119</span>, 10.1073/pnas.2120042119 (2022). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8833196/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35131858" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.2120042119" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Thermodynamics+of+evolution+and+the+origin+of+life&amp;volume=119&amp;publication_year=2022&amp;doi=10.1073/pnas.2120042119&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r87">87. <span class="mixed-citation">Katsnelson M. I., Vanchurin V., Westerhout T., Self-organized criticality in neural networks. arXiv [Preprint] (2021). <a href="https://arxiv.org/abs/2107.03402" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://arxiv.org/abs/2107.03402</a> (Accessed 22 January 2022).</span></div><div class="ref-cit-blk half_rhythm" id="r88">88. <span class="mixed-citation">Katsnelson M. I., Vanchurin V., <span class="ref-title">Quantumness in neural networks</span>. <span class="ref-journal">Found. Phys.</span>
<span class="ref-vol">51</span>, 94 (2021). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Found.+Phys.&amp;title=Quantumness+in+neural+networks&amp;volume=51&amp;publication_year=2021&amp;pages=94&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r89">89. <span class="mixed-citation">Carroll S. M., <span class="ref-journal">Spacetime and Geometry: An Introduction to General Relativity</span> (Addison-Wesley, San Francisco, CA, 2004). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Spacetime+and+Geometry:+An+Introduction+to+General+Relativity&amp;publication_year=2004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r90">90. <span class="mixed-citation">Bagrov A. A., Iakovlev I. A., Iliasov A. A., Katsnelson M. I., Mazurenko V. V., <span class="ref-title">Multiscale structural complexity of natural patterns</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">117</span>, 30241&#x02013;30251 (2020). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7720216/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33208537" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Multiscale+structural+complexity+of+natural+patterns&amp;volume=117&amp;publication_year=2020&amp;pages=30241-30251&amp;pmid=33208537&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r91">91. <span class="mixed-citation">Haken H., <span class="ref-journal">Synergetics, an Introduction: Nonequilibrium Phase Transitions and Self-Organization in Physics, Chemistry, and Biology</span> (Springer, New York, NY, 1983). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Synergetics,+an+Introduction:+Nonequilibrium+Phase+Transitions+and+Self-Organization+in+Physics,+Chemistry,+and+Biology&amp;publication_year=1983&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r92">92. <span class="mixed-citation">Haken H., <span class="ref-journal">Advanced Synergetics: Instability Hierarchies of Self-Organizing Systems and Devices</span> (Springer, New York, NY, 1994). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Advanced+Synergetics:+Instability+Hierarchies+of+Self-Organizing+Systems+and+Devices&amp;publication_year=1994&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r93">93. <span class="mixed-citation">Prigogine I. R., Stengers I., <span class="ref-journal">Order out of Chaos</span> (Bantam, London, United Kingdom, 1984). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Order+out+of+Chaos&amp;publication_year=1984&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r94">94. <span class="mixed-citation">Mezard M., Parisi G., Virasoro M. A., Eds., <span class="ref-journal">Spin Glass Theory and Beyond</span> (World Scientific, Singapore, 1987). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Spin+Glass+Theory+and+Beyond&amp;publication_year=1987&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r95">95. <span class="mixed-citation">Kolmus A., Katsnelson M. I., Khajetoorians A. A., Kappen H. J., <span class="ref-title">Atom-by-atom construction of attractors in a tunable finite size spin array</span>. <span class="ref-journal">New J. Phys.</span>
<span class="ref-vol">22</span>, 023038 (2020). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=New+J.+Phys.&amp;title=Atom-by-atom+construction+of+attractors+in+a+tunable+finite+size+spin+array&amp;volume=22&amp;publication_year=2020&amp;pages=023038&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r96">96. <span class="mixed-citation">Bak P., <span class="ref-journal">How Nature Works. The Science of Self-Organized Criticality</span> (Springer, New York, NY, 1996). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=How+Nature+Works.+The+Science+of+Self-Organized+Criticality&amp;publication_year=1996&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r97">97. <span class="mixed-citation">Bak P., Tang C., Wiesenfeld K., <span class="ref-title">Self-organized criticality: An explanation of the 1/f noise</span>. <span class="ref-journal">Phys. Rev. Lett.</span>
<span class="ref-vol">59</span>, 381&#x02013;384 (1987). [<a href="https://pubmed.ncbi.nlm.nih.gov/10035754" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Phys.+Rev.+Lett.&amp;title=Self-organized+criticality:+An+explanation+of+the+1/f+noise&amp;volume=59&amp;publication_year=1987&amp;pages=381-384&amp;pmid=10035754&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r98">98. <span class="mixed-citation">Dobzhansky T., <span class="ref-title">Nothing in biology makes sense except in the light of evolution</span>. <span class="ref-journal">Am. Biol. Teach.</span>
<span class="ref-vol">35</span>, 125&#x02013;129 (1973). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+Biol.+Teach.&amp;title=Nothing+in+biology+makes+sense+except+in+the+light+of+evolution&amp;volume=35&amp;publication_year=1973&amp;pages=125-129&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r99">99. <span class="mixed-citation">Copley S. D., Smith E., Morowitz H. J., <span class="ref-title">The origin of the RNA world: Co-evolution of genes and metabolism</span>. <span class="ref-journal">Bioorg. Chem.</span>
<span class="ref-vol">35</span>, 430&#x02013;443 (2007). [<a href="https://pubmed.ncbi.nlm.nih.gov/17897696" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.+Chem.&amp;title=The+origin+of+the+RNA+world:+Co-evolution+of+genes+and+metabolism&amp;volume=35&amp;publication_year=2007&amp;pages=430-443&amp;pmid=17897696&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r100">100. <span class="mixed-citation">Kahana A., Lancet D., <span class="ref-title">Self-reproducing catalytic micelles as nanoscopic protocell precursors</span>. <span class="ref-journal">Nat. Rev. Chem.</span>
<span class="ref-vol">5</span>, 870&#x02013;878 (2021). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Chem.&amp;title=Self-reproducing+catalytic+micelles+as+nanoscopic+protocell+precursors&amp;volume=5&amp;publication_year=2021&amp;pages=870-878&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r101">101. <span class="mixed-citation">Vetsigian K., Woese C., Goldenfeld N., <span class="ref-title">Collective evolution and the genetic code</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. U.S.A.</span>
<span class="ref-vol">103</span>, 10696&#x02013;10701 (2006). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1502294/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16818880" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+U.S.A.&amp;title=Collective+evolution+and+the+genetic+code&amp;volume=103&amp;publication_year=2006&amp;pages=10696-10701&amp;pmid=16818880&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r102">102. <span class="mixed-citation">G&#x000e1;nti T., <span class="ref-journal">Chemoton Theory: Theory of Living Systems</span> (Kluwer Academic, Plenum, New York, NY, 2004). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Chemoton+Theory:+Theory+of+Living+Systems&amp;publication_year=2004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r103">103. <span class="mixed-citation">Szathm&#x000e1;ry E., <span class="ref-title">Life: In search of the simplest cell</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">433</span>, 469&#x02013;470 (2005). [<a href="https://pubmed.ncbi.nlm.nih.gov/15690023" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Life:+In+search+of+the+simplest+cell&amp;volume=433&amp;publication_year=2005&amp;pages=469-470&amp;pmid=15690023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r104">104. <span class="mixed-citation">Zachar I., Fedor A., Szathm&#x000e1;ry E., <span class="ref-title">Two different template replicators coexisting in the same protocell: Stochastic simulation of an extended chemoton model</span>. <span class="ref-journal">PLoS One</span>
<span class="ref-vol">6</span>, e21380 (2011). <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3139576/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21818258" ref="reftype=pubmed&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=Two+different+template+replicators+coexisting+in+the+same+protocell:+Stochastic+simulation+of+an+extended+chemoton+model&amp;volume=6&amp;publication_year=2011&amp;pages=e21380&amp;pmid=21818258&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="r105">105. <span class="mixed-citation">Dicke R. H., <span class="ref-title">Dirac&#x02019;s cosmology and Mach&#x02019;s principle</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">192</span>, 440&#x02013;441 (1961). <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Dirac&#x02019;s+cosmology+and+Mach&#x02019;s+principle&amp;volume=192&amp;publication_year=1961&amp;pages=440-441&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=8833143&amp;issue-id=400840&amp;journal-id=2&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Proceedings of the National Academy of Sciences of the United States of America</span> are provided here courtesy of <strong>National Academy of Sciences</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pubreader-link other_item"><a href="/pmc/articles/PMC8833143/?report=reader" class="sidefm-pmclink">PubReader</a></li>
<li class="pdf-link other_item"><a href="/pmc/articles/PMC8833143/pdf/pnas.202120037.pdf" class="int-view">PDF (692K)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/8833143/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/8833143/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="8833143" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8833143%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="cVDeASjr1dUZ87Mozp5Ut2SC4P2s2Pec3i4hl7Ra9RXJj2djuBVaCqBNtflx6yoE">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8833143%2F&amp;text=Toward%20a%20theory%20of%20evolution%20as%20multilevel%20learning" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC8833143%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8833143/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC8833143/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/35121666/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC8833143/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/35121666/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC8833143/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/8833143/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><meta name="citationexporter" content="backend:'https://api.ncbi.nlm.nih.gov/lit/ctxp/v1/pmc/'" /><script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/jquery.citationexporter.min.js">//</script><link rel="stylesheet" href="https://www.ncbi.nlm.nih.gov/corehtml/pmc/ctxp/citationexporter.css" type="text/css" /><script type="text/javascript" src="/core/mathjax/2.7.9/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC8833143/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
